<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0053)https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/ -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="jmedphys"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC2825001/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Automated medical image segmentation techniques</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="./Automated medical image segmentation techniques_files/jig.min.css"><script type="text/javascript" src="./Automated medical image segmentation techniques_files/jig.min.js.download"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of Medical Physics / Association of Medical Physicists of India"><meta name="citation_title" content="Automated medical image segmentation techniques"><meta name="citation_authors" content="Neeraj Sharma, Lalit M. Aggarwal"><meta name="citation_date" content="Jan-Mar 2010"><meta name="citation_issue" content="1"><meta name="citation_volume" content="35"><meta name="citation_firstpage" content="3"><meta name="citation_doi" content="10.4103/0971-6203.58777"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC2825001/?report=abstract"><meta name="citation_pmid" content="20177565"><meta name="DC.Title" content="Automated medical image segmentation techniques"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Wolters Kluwer -- Medknow Publications"><meta name="DC.Contributor" content="Neeraj Sharma"><meta name="DC.Contributor" content="Lalit M. Aggarwal"><meta name="DC.Date" content="Jan-Mar 2010"><meta name="DC.Identifier" content="10.4103/0971-6203.58777"><meta name="DC.Language" content="en"><meta property="og:title" content="Automated medical image segmentation techniques"><meta property="og:type" content="article"><meta property="og:description" content="Accurate segmentation of medical images is a key step in contouring during radiotherapy planning. Computed topography (CT) and Magnetic resonance (MR) imaging are the most widely used radiographic techniques in diagnosis, clinical studies and treatment ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="./Automated medical image segmentation techniques_files/pmc.min.css" type="text/css"><link rel="stylesheet" href="./Automated medical image segmentation techniques_files/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="./Automated medical image segmentation techniques_files/common.min.js.download">//</script><script type="text/javascript" src="./Automated medical image segmentation techniques_files/NcbiTagServer.min.js.download">//</script><meta name="citationexporter" content="backend:&#39;/pmc/utils/ctxp/&#39;"><script type="text/javascript" src="./Automated medical image segmentation techniques_files/jquery.citationexporter.js.download">//</script><link rel="stylesheet" href="./Automated medical image segmentation techniques_files/citationexporter.css" type="text/css"><script type="text/javascript" src="./Automated medical image segmentation techniques_files/MathJax.js.download"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-jmedphys.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/epub/">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="https://www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="F4FC5A18B88138110000000000F100EB.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="./Automated medical image segmentation techniques_files/4143404.css"><link type="text/css" rel="stylesheet" href="./Automated medical image segmentation techniques_files/4157116.css" media="print"><style type="text/css" media="print">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }.print-log li { list-style-image: url('https://www.ncbi.nlm.nih.gov/stat?jsevent=print&ncbi_app=pmc&ncbi_db=pmc&ncbi_pcid=%2Farticles%2FPMC2825001%2F&ncbi_pdid=article&ncbi_phid=F4FC5A18B88138110000000000F100EB.m_8'); }</style><style type="text/css">.print-log { position:absolute;left:-10000px;top:auto;width:1px;height:1px;overflow:hidden; }</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main-bold; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main-italic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Math-italic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="https://www.ncbi.nlm.nih.gov/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="./Automated medical image segmentation techniques_files/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="https://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="https://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="./Automated medical image segmentation techniques_files/emyncbi.html" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="https://www.ncbi.nlm.nih.gov/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="https://www.ncbi.nlm.nih.gov/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:&#39;pmc-search-autocomplete&#39;,disableUrl:&#39;NcbiSearchBarAutoComplCtrl&#39;" autocomplete="off" data-sbconfig="ds:&#39;no&#39;,pjs:&#39;no&#39;,afs:&#39;yes&#39;" aria-haspopup="true" aria-autocomplete="list" role="textbox"><a class="reset" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" style="visibility: hidden;"><img src="./Automated medical image segmentation techniques_files/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="https://www.ncbi.nlm.nih.gov/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="https://www.ncbi.nlm.nih.gov/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="https://www.ncbi.nlm.nih.gov/pmc/journals/976/">J Med Phys</a></li><li class="issue-page"><a class="navlink" href="https://www.ncbi.nlm.nih.gov/pmc/issues/185087/">v.35(1); Jan-Mar 2010</a></li><li class="accid">PMC2825001</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="./Automated medical image segmentation techniques_files/logo-jmedphys.gif" alt="Logo of jmedphys" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="poly" coords="0,0,500,0,500,21,400,21,400,75,0,75" alt="Home" title="Home" href="http://www.jmp.org.in/" ref="reftype=publisher&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="410,24,500,37" alt="Current issue" title="Current issue" href="http://www.jmp.org.in/currentissue.asp" ref="reftype=publisher&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="410,42,500,53" alt="Instructions" title="Instructions" href="http://www.jmp.org.in/contributors.asp" ref="reftype=publisher&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"><area shape="rect" coords="410,59,500,75" alt="Submit article" title="Submit article" href="http://www.journalonweb.com/jmp" ref="reftype=publisher&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A&amp;rendering-type=normal" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: [&#39;h2&#39;], headingExclude: &#39;:hidden&#39;" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="small"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Med Phys</a></span>. 2010 Jan-Mar; 35(1): 3–14. </span></div><div>  <span class="doi">doi:&nbsp; <a href="https://dx.doi.org/10.4103%2F0971-6203.58777" target="pmc_ext" ref="reftype=other&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI&amp;rendering-type=normal">10.4103/0971-6203.58777</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC2825001</span></div><div class="fm-citation-pmid">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20177565">20177565</a></div></div></div></div></div><h1 class="content-title">Automated medical image segmentation techniques</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Sharma%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20177565" class="affpopup" co-rid="_co_idm140507406237152" co-class="co-affbox">Neeraj Sharma</a><sup></sup><sup></sup> and  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Aggarwal%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20177565" class="affpopup" co-rid="_co_idm140507406524064" co-class="co-affbox">Lalit M. Aggarwal</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140507406237152"><h3 class="no_margin">Neeraj Sharma</h3><p>School of Biomedical Engineering, Institute of Technology, Institute of Medical Sciences, Banaras Hindu University, Varanasi-221 005, UP, India</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Sharma%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20177565">Neeraj Sharma</a></div></div><div id="_co_idm140507406524064"><h3 class="no_margin">Lalit M. Aggarwal</h3><p><sup>1</sup>Department of Radiotherapy and Radiation Medicine, Institute of Medical Sciences, Banaras Hindu University, Varanasi-221 005, UP, India</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Aggarwal%20LM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20177565">Lalit M. Aggarwal</a></div></div></div></div><div class="fm-panel small half_rhythm"><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140507412325200_ai" style="display:none"><div class="fm-affl" lang="en" id="AF0001">School of Biomedical Engineering, Institute of Technology, Institute of Medical Sciences, Banaras Hindu University, Varanasi-221 005, UP, India</div><div class="fm-affl" lang="en" id="AF0002"><sup>1</sup>Department of Radiotherapy and Radiation Medicine, Institute of Medical Sciences, Banaras Hindu University, Varanasi-221 005, UP, India</div><div id="cor1"><strong>Address for correspondence:</strong> Dr. Neeraj Sharma School of Biomedical Engineering, Institute of Technology, Banaras Hindu University, Varanasi-221 005, UP, India E-mail: <a href="mailto:dev@null" data-email="moc.semitaidni@92jareen_re" class="oemail">moc.semitaidni@92jareen_re</a></div></div><div class="togglers"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" class="pmctoggle" rid="idm140507412325200_ai">Author information <span>►</span></a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" class="pmctoggle" rid="idm140507412325200_an">Article notes <span>►</span></a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" class="pmctoggle" rid="idm140507412325200_cpl">Copyright and License information <span>►</span></a> <a href="https://www.ncbi.nlm.nih.gov/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140507412325200_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2009 Apr 25; Revised 2009 Jul 15; Accepted 2009 Aug 24.</div></div><div class="fm-cpl-info fm-panel hide half_rhythm" id="idm140507412325200_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="https://www.ncbi.nlm.nih.gov/pmc/about/copyright/">Copyright</a> © Journal of Medical Physics</div><div class="fm-copyright half_rhythm"><div class="license">This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="small"><div>This article has been <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140507415367776" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140507415367776title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Accurate segmentation of medical images is a key step in contouring during radiotherapy planning. Computed topography (CT) and Magnetic resonance (MR) imaging are the most widely used radiographic techniques in diagnosis, clinical studies and treatment planning. This review provides details of automated segmentation methods, specifically discussed in the context of CT and MR images. The motive is to discuss the problems encountered in segmentation of CT and MR images, and the relative merits and limitations of methods currently available for segmentation of medical images.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Artificial intelligence techniques, computed tomography, magnetic resonance imaging, medical images artifacts, segmentation</span></div></div><div id="sec1-1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-1title">Introduction</h2><p id="__p2" class="p p-first">With increasing use of Computed topography (CT) and Magnetic resonance (MR) imaging for diagnosis, treatment planning and clinical studies, it has become almost compulsory to use computers to assist radiological experts in clinical diagnosis, treatment planning. Reliable algorithms are required for the delineation of anatomical structures and other regions of interest (ROI). The goals of computer-aided diagnosis (CAD) are:</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>To automate the process so that large number of cases can be handled with the same accuracy i.e. the results are not affected as a result of fatigue, data overload or missing manual steps.</div></li><li><div>To achieve fast and accurate results. Very high-speed computers are, now, available at modest costs, speeding up computer-based processing in the medical field.</div></li><li><div>To support faster communication, wherein patient care can be extended to remote areas using information technology.</div></li></ul><p id="__p6">The techniques available for segmentation of medical images are specific to application, imaging modality and type of body part to be studied. For example, requirements of brain segmentation are different from those of thorax. The artifacts, which affect the brain image, are different - partial volume effect is more prominent in brain while in the thorax region it is motion artifact which is more prominent. Thus while selecting a segmentation algorithm one is required to consider all these aspects. The problems common to both CT and MR medical images are:</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>Partial volume effect</div></li><li><div>Different artifacts: example motion artifacts, ring artifacts, etc and</div></li><li><div>Noise due to sensors and related electronic system.</div></li></ul><p id="__p10" class="p p-last">There is no universal algorithm for segmentation of every medical image. Each imaging system has its own specific limitations. For example, in MR imaging (MRI) one has to take care of bias field noise (intensity in-homogeneities in the RF field). Of course, some methods are more general as compared to specialized algorithms and can be applied to a wider range of data. A brief survey of three generations of medical image segmentation techniques has been provided by D.J. Withey and Z.J. Koles.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT1" rid="CIT1" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">1</a>]</p></div><div id="sec1-2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-2title">Medical Imaging Modalities</h2><p id="__p11" class="p p-first">Medical imaging is performed in various modalities, such as MRI, CT, ultrasound, positron emission tomography (PET), etc. In the present review, we are focusing primarily on the segmentation of MR and CT images only.</p><div id="sec2-1" class="sec"><div id="sec3-1" class="sec sec-first-last"><p></p><h4 class="inline">Magnetic Resonance Imaging: </h4><p id="__p12" class="p p-first-last">MR imaging is the most widely used technique in the field of radio imaging.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT2" rid="CIT2" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">2</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT3" rid="CIT3" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">3</a>] MR is a dynamic and flexible technology that allows achieving variable image contrast by using different pulse sequences and by changing the imaging parameters corresponding to longitudinal relaxation time (T1), and transverse relaxation time (T2), and signal intensities on T1 and T2 weighted images relate to specific tissue characteristics. The contrast on MR image is a factor dependent on pulse sequence parameters. The most common pulse sequences are T1- weighted and T2-weighted spin-echo sequences. MR imaging of the body is performed to get the structural details of brain, liver, chest, abdomen and pelvis which helps in diagnosis or monitoring the treatment.</p></div></div><div id="sec2-2" class="sec"><h3>Brain MR Imaging:</h3><p id="__p13" class="p p-first">MR is generally more sensitive in detecting brain abnormalities during the early stages of disease, and is excellent in early detection of cases of cerebral infarction, brain tumors, or infections. MR is particularly useful in detecting white matter disease, such as multiple sclerosis, progressive multifocal leukoencephalopathy, leukodystrophy, and post-infectious encephalitis. In contrast, CT scan fails to detect white matter abnormalities. In case of MR images of the brain, the primary determinants of signal intensity and contrast are the T1 and T2 relaxation times. The contrast is distinctly different on T1 and T2-weighted images. Also, brain pathologies have some common signal characteristics. Pathologic lesions can be separated into five major groups by their specific signal characteristics on the two basic images: T2- weighted, and T1-weighted [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/table/T0001/" target="table" class="fig-table-link figpopup" rid-figpopup="T0001" rid-ob="ob-T0001" co-legend-rid=""><span>Table 1</span></a>].</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T0001"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p14">Type of Pathology and its Contrast in T1 and T2 Weighted Image</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140507408266640" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" rowspan="1" colspan="1"><em>Pathology</em></th><th align="center" rowspan="1" colspan="1"><em>Contrast in T2 weighted image</em></th><th align="center" rowspan="1" colspan="1"><em>Contrast in T1 weighted image</em></th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">Solid Mass</td><td align="center" rowspan="1" colspan="1">Bright</td><td align="center" rowspan="1" colspan="1">Dark</td></tr><tr><td align="left" rowspan="1" colspan="1">Fat</td><td align="center" rowspan="1" colspan="1">Dark</td><td align="center" rowspan="1" colspan="1">Bright</td></tr><tr><td align="left" rowspan="1" colspan="1">Cyst</td><td align="center" rowspan="1" colspan="1">Bright</td><td align="center" rowspan="1" colspan="1">Dark</td></tr><tr><td align="left" rowspan="1" colspan="1">Acute and chronic blood</td><td align="center" rowspan="1" colspan="1">Dark</td><td align="center" rowspan="1" colspan="1">Gray</td></tr><tr><td align="left" rowspan="1" colspan="1">Sub acute blood</td><td align="center" rowspan="1" colspan="1">Bright</td><td align="center" rowspan="1" colspan="1">Bright</td></tr></tbody></table></div><div id="largeobj_idm140507408266640" class="largeobj-link align_right" style="display: none"><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/table/T0001/?report=objectonly">Open in a separate window</a></div></div></div><div id="sec2-3" class="sec"><h3>MR Liver Imaging:</h3><p id="__p15" class="p p-first">MR provides outstanding intrinsic soft contrast that can enhance subtle differences between normal and pathologic tissues and tissues of different histologic subtypes. Nonionizing radiation is used and MRI contrast agents are not nephro-toxic. MRI images may be acquired with multiplanar capabilities which are especially useful in depicting various anatomic relationships. MRI system is specifically used in characterization of metastases and primary liver tumors e.g., benign lesions such as focal nodular hyperplasia (FNH), adenoma, hemangioma and malignant lesions (cancer) such as hepatocellular carcinomas (HCC).</p><p id="__p16"><strong>Chest MRI</strong> is used to detect following disorders: thymus tumor, lung masses, esophageal mass, other masses (aggregations of cells) or tumors of the chest, abnormal lymph nodes, swollen glands and enlarged lymph nodes in any location of the chest, staging of tumors including invasion of blood vessels, alveolar bullae (COPD), bronchial abnormalities, bronchiectasis, cystic lung lesions, pleural abnormalities, including thickening or pleural effusion, abnormal pulmonary vessels, aortic stenosis, etc.</p><p id="__p17"><strong>Abdominal MRI</strong> may reveal many medical conditions, including: abscess, acute tubular necrosis, adrenal masses, cancer, enlarged spleen or liver, gallbladder or bile duct problems, gallstones, bile duct stones, hemangiomas, kidney infection, kidney damage, lymphadenopathy, obstructed venacava, pancreatic cancer, tumor of the gallbladder, abdominal aortic aneurysm, ovarian cancer, etc.</p><p id="__p18">MR Angiography is used to detect blockages or enlargements of blood vessels, including the aorta, renal arteries, and arteries in the legs, renal arterial obstruction, renal vein thrombosis, etc.</p><p id="__p19">The following artifacts are present in MR imaging:</p><ol class="enumerated" style="list-style-type:decimal"><!--
list-behavior=enumerated
prefix-word=
mark-type=decimal
max-label-size=0
--><li><div>Partial Volume</div></li><li><div>RF Noise</div></li><li><div>Intensity inhomogeneity</div></li><li><div>Gradient</div></li><li><div>Motion</div></li><li><div>Wrap Around</div></li><li><div>Gibbs Ringing</div></li><li><div>Susceptibility</div></li></ol><p id="__p28" class="p">These MR imaging artifacts are shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0001" rid-ob="ob-F0001" co-legend-rid="lgnd_F0001"><span>figure 1</span></a> respectively.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F0001" co-legend-rid="lgnd_F0001"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0001/" target="figure" rid-figpopup="F0001" rid-ob="ob-F0001"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140507411070896" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g001.jpg" src="./Automated medical image segmentation techniques_files/JMP-35-3-g001.jpg"></div></a><div id="largeobj_idm140507411070896" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0001/" target="figure" rid-figpopup="F0001" rid-ob="ob-F0001"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0001/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F0001"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0001/" target="figure" rid-figpopup="F0001" rid-ob="ob-F0001">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p29">Artifacts in MR Imaging</p></div></div></div><div id="sec3-2" class="sec"><p></p><h4 class="inline">The main advantages of MR imaging system are:. </h4><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>It has an excellent capability for soft tissue imaging</div></li><li><div>It has very high resolution of the order of 1mm cubic voxels</div></li><li><div>It has high signal to noise ratio</div></li><li><div>Multi channel images with variable contrast can be achieved by using different pulse sequences; this can be further utilized for segmenting and classifying different structures.</div></li></ul></div><div id="sec3-3" class="sec"><p></p><h4 class="inline">Disadvantages of MR imaging: </h4><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>MR acquisition takes considerably longer time as compared to CT and</div></li><li><div>In case of MR it is more difficult to obtain uniform image quality.</div></li></ul></div><div id="sec3-4" class="sec sec-last"><p></p><h4 class="inline">Computed Tomography Imaging: </h4><p id="__p36" class="p p-first">The word tomography is derived from two Greek words; tomos, which means slice or section, and graphia, which means description. CT scan is an imaging modality which uses X-rays to obtain structural and functional information about the human body. The CT image is the reconstructed image and is reconstructed on the basis of X-ray absorption profile. X-rays are electromagnetic waves and used in diagnosis based on its property that all matters and tissues differ in their ability to absorb X-rays.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT2" rid="CIT2" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">2</a>] Dense tissues such as the bones appear white on a CT film while soft tissues such as the brain or liver appear gray. The cavities filled with air such as lungs appear black. CT performs better in cases of trauma and emergent situations. It provides better bone detail and has high sensitivity for acute hemorrhage. CT has become an important tool in medical imaging to supplement X-rays, medical ultrasonography (USG) and MR imaging. Although it is still quite expensive, it is the gold standard in the diagnosis of a large number of different disease entities. It is, more recently, being used in early screening of diseases, for example CT colonography for patients with a high risk of colon cancer.</p><p id="__p37" class="p p-last">CT scans are particularly used in imaging and the diagnosis of following body parts: brain, liver, chest, abdomen and pelvis, spine and also for CT based angiography.</p></div></div><div id="sec2-4" class="sec"><h3>Brain CT Imaging:</h3><p id="__p38" class="p p-first">In case of brain imaging, CT scans are typically used to detect: bleeding, brain damage and skull fracture in patients with head injuries; bleeding caused by a ruptured or leaking aneurysm in a patient with a sudden severe headache, blood clot or bleeding within the brain shortly after a patient exhibits symptoms of a stroke, brain tumors, cyst, diseases related to malformations of the skull, enlarged brain cavities (ventricles).</p><p id="__p39" class="p p-last">CT scanning is fast and simple, provides more detailed information on head injuries, and stroke; can reveal internal injuries and bleeding quick enough to help save lives in emergency cases.</p></div><div id="sec2-5" class="sec"><h3>Liver CT Imaging:</h3><p id="__p40" class="p p-first-last">In case of liver imaging, CT is the most commonly used imaging technique for evaluation of hepatic lesions. Large hepatocellular carcinomas tend to be heterogeneous, and may demonstrate a typical mosaic appearance on CT. There are relative advantages and disadvantages of both hepatic MR and CT. In general, CT is less costly than MR, more readily available, and most radiologists and many referring physicians have a relatively high degree of confidence in looking at CT images. Some studies, however, have found that CT is less sensitive and specific than MR for detection and characterization of focal hepatic disease.</p></div><div id="sec2-6" class="sec"><h3>Chest Imaging:</h3><p id="__p41" class="p p-first-last">Chest CT is used to detect: tumors in lungs, pneumonia, tuberculosis, emphysema, diffuse interstitial lung diseases, inflammation or other diseases of pleura, the membrane covering the lungs.</p></div><div id="sec2-7" class="sec"><h3>Abdomen and Pelvis CT Imaging:</h3><p id="__p42" class="p p-first-last">It is used to detect: abscesses in abdomen, inflamed colon, cancers of (i) colon, (ii) liver and (iii) pancreas, pancreatitis, lymphoma, diverticulitis, appendicitis.</p></div><div id="sec2-8" class="sec"><h3>Spine CT Imaging:</h3><p id="__p43" class="p p-first">It is used to detect various types of tumors in vertebral column, herniated inter-vertebral disk, fractures and other injuries and measure bone density and level of osteoporosis.</p><p id="__p44">CT based angiography is used to identify a small aneurysm or arteriovenous malformation inside the brain, detect thrombosis in veins, atherosclerotic disease in the carotid artery and indicate disease in renal artery.</p><p id="__p45">The artifacts present in CT imaging are:[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT4" rid="CIT4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966066" role="button" aria-expanded="false" aria-haspopup="true">4</a>]</p><p id="__p46">1) partial volume effect [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0001/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0001" rid-ob="ob-F0001" co-legend-rid="lgnd_F0001"><span>Figure 1a</span></a>], 2) streak artifacts, 3) motion artifacts, 4) beam hardening artifacts, 5) ring artifacts, and 6) bloom artifacts.</p><p id="__p47" class="p">These artifacts are shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0002/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0002" rid-ob="ob-F0002" co-legend-rid="lgnd_F0002"><span>Figure 2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F0002" co-legend-rid="lgnd_F0002"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0002/" target="figure" rid-figpopup="F0002" rid-ob="ob-F0002"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140507406442336" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g002.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g002.jpg" src="./Automated medical image segmentation techniques_files/JMP-35-3-g002.jpg"></div></a><div id="largeobj_idm140507406442336" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0002/" target="figure" rid-figpopup="F0002" rid-ob="ob-F0002"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0002/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F0002"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0002/" target="figure" rid-figpopup="F0002" rid-ob="ob-F0002">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p48">Examples of CT Artifacts: (A) Streak (B) Motion (C) Beam-hardening (D-E) Ring (F) Bloom [4]</p></div></div></div><div id="sec3-5" class="sec"><p></p><h4 class="inline">Advantages of CT imaging include: </h4><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>Less expense and wide availablity</div></li><li><div>High spatial resolution with modern multi-slice scanners</div></li><li><div>Short scan time</div></li><li><div>Higher sensitivity than MR for sub-arachnoids hemorrhage</div></li><li><div>Higher sensitivity in detecting intra-cranial calcifications</div></li></ul></div><div id="sec3-6" class="sec sec-last"><p></p><h4 class="inline">Disadvantages of CT imaging system are: </h4><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>Inferior soft tissue contrast compared to MRI as it is X-ray-based</div></li><li><div>Radiation exposure</div></li></ul><p id="__p56" class="p p-last">Despite the disadvantages, CT scans are extensively used in the radiographic study of brain, liver and thorax.</p></div></div><div id="sec2-9" class="sec sec-last"><h3>Representation of Medical Images:</h3><p id="__p57" class="p p-first-last">Images are presented in 2-D as well as in 3-D domain. In the 2-D domain each element is called pixel, while in 3-D domain it is called voxel. In some cases we represent 3-D images as a sequential series of 2-D slices. The advantages associated with this type of representation include requirement of lower computational complexity and lesser memory.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT5" rid="CIT5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577496" role="button" aria-expanded="false" aria-haspopup="true">5</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT6" rid="CIT6" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">6</a>]</p></div></div><div id="sec1-3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-3title">Segmentation</h2><p id="__p58" class="p p-first">Segmentation is the process dividing an image into regions with similar properties such as gray level, color, texture, brightness, and contrast.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT7" rid="CIT7" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">7</a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT9" rid="CIT9" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">9</a>] The role of segmentation is to subdivide the objects in an image; in case of medical image segmentation the aim is to:</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>Study anatomical structure</div></li><li><div>Identify Region of Interest i.e. locate tumor, lesion and other abnormalities</div></li><li><div>Measure tissue volume to measure growth of tumor (also decrease in size of tumor with treatment)</div></li><li><div>Help in treatment planning prior to radiation therapy; in radiation dose calculation</div></li></ul><p id="__p63">Automatic segmentation of medical images is a difficult task as medical images are complex in nature and rarely have any simple linear feature. Further, the output of segmentation algorithm is affected due to</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>partial volume effect.</div></li><li><div>intensity inhomogeneity</div></li><li><div>presence of artifacts</div></li><li><div>closeness in gray level of different soft tissue</div></li></ul><p id="__p68">Artifacts present in MR and CT images can be divided into three categories on the basis of image processing technique needed to rectify them: (i) artifacts needing appropriate filtering technique. For example, noise artifact, susceptibility artifact and presence of nonsharp edges in the image (ii) artifact needing appropriate image restoration techniques for example motion artifacts and (iii) artifact needing specific algorithm are; partial volume, intensity inhomogeneity.</p><p id="__p69" class="p">Although a number of algorithms have been proposed in the field of medical image segmentation, medical image segmentation continues to be a complex and challenging problem. Different researchers have done the classification of segmentation techniques in one or another way.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT1" rid="CIT1" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">1</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT10" rid="CIT10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577494" role="button" aria-expanded="false" aria-haspopup="true">10</a>] At present, from the medical image processing point of view we have done the classification of segmentation techniques on the basis of gray level based and textural feature based techniques. Further, we consider artificial intelligence as tools to optimize these basic techniques to achieve accurate segmentation results. Thus, the broad classification of techniques available for segmentation of an image classified into two classes is as follows:</p><div id="sec2-10" class="sec"><h3>Methods based on gray Level features</h3><ol class="enumerated" style="list-style-type:lower-alpha"><!--
list-behavior=enumerated
prefix-word=
mark-type=lower-alpha
max-label-size=0
--><li><div>Amplitude segmentation based on histogram features[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT11" rid="CIT11" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">11</a>]</div></li><li><div>Edge based segmentation</div></li><li><div>Region based segmentation[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT12" rid="CIT12" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">12</a>]</div></li></ol></div><div id="sec2-11" class="sec"><h3>Methods based on texture features[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT13" rid="CIT13" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">13</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT14" rid="CIT14" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">14</a>]</h3></div><div id="sec2-12" class="sec"><h3>Method based on gray level features</h3></div><div id="sec2-13" class="sec"><h3>Amplitude segmentation based on histogram features</h3><p id="__p73" class="p p-first">This includes segmentation of an image based on thresholding of histogram features and gray level thresholding is perhaps the simplest example of this technique. This is particularly suitable for an image with region or object of uniform brightness placed against a back ground of different gray level, A threshold can be applied to segment the object and background. Mathematically the threshold can be defined as follows.</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="M1" overflow="scroll" role="math" style="width: 11.394em; display: inline-block;"><span style="display: inline-block; position: relative; width: 9.176em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(1.817em, 1009.02em, 4.539em, -999.997em); top: -3.425em; left: 0.003em;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="mi" id="MathJax-Span-4" style="font-family: MathJax_Math-italic;">r</span><span class="mrow" id="MathJax-Span-5"><span class="mi" id="MathJax-Span-6" style="font-family: MathJax_Math-italic;">i</span><span class="mo" id="MathJax-Span-7" style="font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-8" style="font-family: MathJax_Math-italic; padding-left: 0.154em;">j</span></span><span class="mo" id="MathJax-Span-9" style="font-family: MathJax_Main; padding-left: 0.305em;">=</span><span class="mrow" id="MathJax-Span-10" style="padding-left: 0.305em;"><span class="mo" id="MathJax-Span-11" style="vertical-align: 0.003em;"><span style="font-family: MathJax_Size3;">{</span></span><span class="mrow" id="MathJax-Span-12"><span class="mtable" id="MathJax-Span-13" style="padding-right: 0.154em; padding-left: 0.154em;"><span style="display: inline-block; position: relative; width: 5.093em; height: 0px;"><span style="position: absolute; clip: rect(2.523em, 1000.46em, 4.942em, -999.997em); top: -3.979em; left: 0.003em;"><span style="display: inline-block; position: relative; width: 0.507em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.41em, 4.136em, -999.997em); top: -4.635em; left: 50%; margin-left: -0.249em;"><span class="mtd" id="MathJax-Span-14"><span class="mrow" id="MathJax-Span-15"><span class="mn" id="MathJax-Span-16" style="font-family: MathJax_Main;">1</span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.178em, 1000.46em, 4.136em, -999.997em); top: -3.223em; left: 50%; margin-left: -0.249em;"><span class="mtd" id="MathJax-Span-27"><span class="mrow" id="MathJax-Span-28"><span class="mn" id="MathJax-Span-29" style="font-family: MathJax_Main;">0</span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.472em, 1003.78em, 5.093em, -999.997em); top: -3.979em; left: 1.313em;"><span style="display: inline-block; position: relative; width: 3.783em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1003.78em, 4.337em, -999.997em); top: -4.635em; left: 50%; margin-left: -1.913em;"><span class="mtd" id="MathJax-Span-17"><span class="mrow" id="MathJax-Span-18"><span class="mrow" id="MathJax-Span-19"><span class="mi" id="MathJax-Span-20" style="font-family: MathJax_Math-italic;">p</span><span class="mrow" id="MathJax-Span-21"><span class="mi" id="MathJax-Span-22" style="font-family: MathJax_Math-italic;">i</span><span class="mo" id="MathJax-Span-23" style="font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-24" style="font-family: MathJax_Math-italic; padding-left: 0.154em;">j</span></span><span class="mo" id="MathJax-Span-25" style="font-family: MathJax_Main; padding-left: 0.305em;">≥</span><span class="mi" id="MathJax-Span-26" style="font-family: MathJax_Math-italic; padding-left: 0.305em;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.103em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.178em, 1003.78em, 4.337em, -999.997em); top: -3.223em; left: 50%; margin-left: -1.913em;"><span class="mtd" id="MathJax-Span-30"><span class="mrow" id="MathJax-Span-31"><span class="mrow" id="MathJax-Span-32"><span class="mi" id="MathJax-Span-33" style="font-family: MathJax_Math-italic;">p</span><span class="mrow" id="MathJax-Span-34"><span class="mi" id="MathJax-Span-35" style="font-family: MathJax_Math-italic;">i</span><span class="mo" id="MathJax-Span-36" style="font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-37" style="font-family: MathJax_Math-italic; padding-left: 0.154em;">j</span></span><span class="mo" id="MathJax-Span-38" style="font-family: MathJax_Main; padding-left: 0.305em;">≺</span><span class="mi" id="MathJax-Span-39" style="font-family: MathJax_Math-italic; padding-left: 0.305em;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.103em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.43em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.247em; border-left: 0px solid; width: 0px; height: 3.128em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" overflow="scroll"><mrow><mi>r</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mo>=</mo><mrow><mo>{</mo><mrow><mtable><mtr><mtd><mn>1</mn></mtd><mtd><mrow><mi>p</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mo>≥</mo><mi>T</mi></mrow></mtd></mtr><mtr><mtd><mn>0</mn></mtd><mtd><mrow><mi>p</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mo>≺</mo><mi>T</mi></mrow></mtd></mtr></mtable></mrow></mrow></mrow></math></script></div><div class="l">(1)</div></div><p></p><p id="__p75">Where <em>r<sub>i, j</sub></em> is the resulting pixel at co-ordinate <em>(i, j), p<sub>i, j</sub></em> is the pixel of input image and <em>T</em> is the value of threshold.</p><p id="__p76"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#FD1" rid="FD1" class=" disp-formula">Equation 1</a> gives good results for segmentation of image with bi-modal histogram and fails in the case of an image with multi-modal histogram. Thresholding operation, defined by <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#FD1" rid="FD1" class=" disp-formula">equation 1</a> is very basic and simple, and works well only when the object and background have uniform brightness of distinct gray level values respectively. This simple threshold operation does not work well at segmentation of images with multiple objects each having distinct gray level value varying over a band of values. To overcome this limitation, band thresholding based multiple thresholding operation is applied as follows:</p><div class="disp-formula" id="FD2"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-2-Frame" tabindex="0" style=""><nobr><span class="math" id="M2" overflow="scroll" role="math" style="width: 15.023em; display: inline-block;"><span style="display: inline-block; position: relative; width: 12.099em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(-1.762em, 1011.95em, 5.597em, -999.997em); top: -2.165em; left: 0.003em;"><span class="mrow" id="MathJax-Span-41"><span class="mrow" id="MathJax-Span-42"><span class="mtable" id="MathJax-Span-43" style="padding-right: 0.154em; padding-left: 0.154em;"><span style="display: inline-block; position: relative; width: 11.797em; height: 0px;"><span style="position: absolute; clip: rect(3.984em, 1001.26em, 10.789em, -999.997em); top: -7.558em; left: 0.003em;"><span style="display: inline-block; position: relative; width: 1.263em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1001.26em, 4.438em, -999.997em); top: -7.004em; left: 50%; margin-left: -0.602em;"><span class="mtd" id="MathJax-Span-44"><span class="mrow" id="MathJax-Span-45"><span class="mrow" id="MathJax-Span-46"><span class="msub" id="MathJax-Span-47"><span style="display: inline-block; position: relative; width: 1.263em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-48" style="font-family: MathJax_Math-italic;">r</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.456em;"><span class="mrow" id="MathJax-Span-49"><span class="mi" id="MathJax-Span-50" style="font-size: 70.7%; font-family: MathJax_Math-italic;">i</span><span class="mo" id="MathJax-Span-51" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-52" style="font-size: 70.7%; font-family: MathJax_Math-italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.833em, 1000em, 4.136em, -999.997em); top: -5.441em; left: 50%; margin-left: 0.003em;"><span class="mtd" id="MathJax-Span-73"><span class="mrow" id="MathJax-Span-74"><span class="mrow" id="MathJax-Span-75"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.833em, 1000em, 4.136em, -999.997em); top: -3.929em; left: 50%; margin-left: 0.003em;"><span class="mtd" id="MathJax-Span-96"><span class="mrow" id="MathJax-Span-97"><span class="mrow" id="MathJax-Span-98"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.833em, 1000em, 4.136em, -999.997em); top: -2.417em; left: 50%; margin-left: 0.003em;"><span class="mtd" id="MathJax-Span-119"><span class="mrow" id="MathJax-Span-120"><span class="mrow" id="MathJax-Span-121"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.833em, 1000em, 4.136em, -999.997em); top: -0.905em; left: 50%; margin-left: 0.003em;"><span class="mtd" id="MathJax-Span-141"><span class="mrow" id="MathJax-Span-142"><span class="mrow" id="MathJax-Span-143"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 7.563em;"></span></span><span style="position: absolute; clip: rect(3.984em, 1009.73em, 11.091em, -999.997em); top: -7.81em; left: 2.069em;"><span style="display: inline-block; position: relative; width: 9.73em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1009.68em, 4.438em, -999.997em); top: -7.004em; left: 50%; margin-left: -4.836em;"><span class="mtd" id="MathJax-Span-53"><span class="mrow" id="MathJax-Span-54"><span class="mrow" id="MathJax-Span-55"><span class="mo" id="MathJax-Span-56" style="font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-57" style="font-family: MathJax_Main; padding-left: 0.305em;">1</span><span class="msub" id="MathJax-Span-58"><span style="display: inline-block; position: relative; width: 2.876em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1002.37em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-59"><span class="mtext" id="MathJax-Span-60" style="font-family: MathJax_Main;">&nbsp;for&nbsp;T</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 2.422em;"><span class="mn" id="MathJax-Span-61" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-62" style="font-family: MathJax_Main; padding-left: 0.305em;">&lt;</span><span class="msub" id="MathJax-Span-63" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 1.313em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.51em, 4.337em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-64" style="font-family: MathJax_Math-italic;">p</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mrow" id="MathJax-Span-65"><span class="mi" id="MathJax-Span-66" style="font-size: 70.7%; font-family: MathJax_Math-italic;">i</span><span class="mo" id="MathJax-Span-67" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-68" style="font-size: 70.7%; font-family: MathJax_Math-italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-69" style="font-family: MathJax_Main; padding-left: 0.305em;">≤</span><span class="msub" id="MathJax-Span-70" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 1.162em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.71em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mtext" id="MathJax-Span-71" style="font-family: MathJax_Main;">T</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.708em;"><span class="mn" id="MathJax-Span-72" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.128em, 1009.68em, 4.438em, -999.997em); top: -5.441em; left: 50%; margin-left: -4.836em;"><span class="mtd" id="MathJax-Span-76"><span class="mrow" id="MathJax-Span-77"><span class="mrow" id="MathJax-Span-78"><span class="mo" id="MathJax-Span-79" style="font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-80" style="font-family: MathJax_Main; padding-left: 0.305em;">2</span><span class="msub" id="MathJax-Span-81"><span style="display: inline-block; position: relative; width: 2.876em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1002.37em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-82"><span class="mtext" id="MathJax-Span-83" style="font-family: MathJax_Main;">&nbsp;for&nbsp;T</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 2.422em;"><span class="mn" id="MathJax-Span-84" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-85" style="font-family: MathJax_Main; padding-left: 0.305em;">&lt;</span><span class="msub" id="MathJax-Span-86" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 1.313em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.51em, 4.337em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-87" style="font-family: MathJax_Math-italic;">p</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mrow" id="MathJax-Span-88"><span class="mi" id="MathJax-Span-89" style="font-size: 70.7%; font-family: MathJax_Math-italic;">i</span><span class="mo" id="MathJax-Span-90" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-91" style="font-size: 70.7%; font-family: MathJax_Math-italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-92" style="font-family: MathJax_Main; padding-left: 0.305em;">≤</span><span class="msub" id="MathJax-Span-93" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 1.162em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.71em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mtext" id="MathJax-Span-94" style="font-family: MathJax_Main;">T</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.708em;"><span class="mn" id="MathJax-Span-95" style="font-size: 70.7%; font-family: MathJax_Main;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.128em, 1009.68em, 4.438em, -999.997em); top: -3.929em; left: 50%; margin-left: -4.836em;"><span class="mtd" id="MathJax-Span-99"><span class="mrow" id="MathJax-Span-100"><span class="mrow" id="MathJax-Span-101"><span class="mo" id="MathJax-Span-102" style="font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-103" style="font-family: MathJax_Main; padding-left: 0.305em;">3</span><span class="msub" id="MathJax-Span-104"><span style="display: inline-block; position: relative; width: 2.876em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1002.37em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-105"><span class="mtext" id="MathJax-Span-106" style="font-family: MathJax_Main;">&nbsp;for&nbsp;T</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 2.422em;"><span class="mn" id="MathJax-Span-107" style="font-size: 70.7%; font-family: MathJax_Main;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-108" style="font-family: MathJax_Main; padding-left: 0.305em;">&lt;</span><span class="msub" id="MathJax-Span-109" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 1.313em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.51em, 4.337em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-110" style="font-family: MathJax_Math-italic;">p</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mrow" id="MathJax-Span-111"><span class="mi" id="MathJax-Span-112" style="font-size: 70.7%; font-family: MathJax_Math-italic;">i</span><span class="mo" id="MathJax-Span-113" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-114" style="font-size: 70.7%; font-family: MathJax_Math-italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-115" style="font-family: MathJax_Main; padding-left: 0.305em;">≤</span><span class="msub" id="MathJax-Span-116" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 1.162em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.71em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mtext" id="MathJax-Span-117" style="font-family: MathJax_Main;">T</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.708em;"><span class="mn" id="MathJax-Span-118" style="font-size: 70.7%; font-family: MathJax_Main;">4</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.128em, 1009.73em, 4.438em, -999.997em); top: -2.417em; left: 50%; margin-left: -4.887em;"><span class="mtd" id="MathJax-Span-122"><span class="mrow" id="MathJax-Span-123"><span class="mrow" id="MathJax-Span-124"><span class="mo" id="MathJax-Span-125" style="font-family: MathJax_Main;">=</span><span class="msub" id="MathJax-Span-126" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 3.38em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1002.93em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-127"><span class="mtext" id="MathJax-Span-128" style="font-family: MathJax_Main;">k&nbsp;for&nbsp;T</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 2.926em;"><span class="mtext" id="MathJax-Span-129" style="font-size: 70.7%; font-family: MathJax_Main;">k</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-130" style="font-family: MathJax_Main; padding-left: 0.305em;">&lt;</span><span class="msub" id="MathJax-Span-131" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 1.313em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.51em, 4.337em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-132" style="font-family: MathJax_Math-italic;">p</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mrow" id="MathJax-Span-133"><span class="mi" id="MathJax-Span-134" style="font-size: 70.7%; font-family: MathJax_Math-italic;">i</span><span class="mo" id="MathJax-Span-135" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-136" style="font-size: 70.7%; font-family: MathJax_Math-italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-137" style="font-family: MathJax_Main; padding-left: 0.305em;">≤</span><span class="msub" id="MathJax-Span-138" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 1.162em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.71em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mtext" id="MathJax-Span-139" style="font-family: MathJax_Main;">T</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.708em;"><span class="mtext" id="MathJax-Span-140" style="font-size: 70.7%; font-family: MathJax_Main;">k</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.128em, 1005.95em, 4.136em, -999.997em); top: -0.905em; left: 50%; margin-left: -2.971em;"><span class="mtd" id="MathJax-Span-144"><span class="mrow" id="MathJax-Span-145"><span class="mrow" id="MathJax-Span-146"><span class="mo" id="MathJax-Span-147" style="font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-148" style="font-family: MathJax_Main; padding-left: 0.305em;">0</span><span class="mtext" id="MathJax-Span-149" style="font-family: MathJax_Main;">&nbsp;otherwise</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 7.815em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.17em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -4.122em; border-left: 0px solid; width: 0px; height: 8.941em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M2" overflow="scroll"><mrow><mtable><mtr><mtd><mrow><msub><mi>r</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow></mtd><mtd><mrow><mo>=</mo><mn>1</mn><msub><mrow><mtext> for T</mtext></mrow><mn>1</mn></msub><mo>&lt;</mo><msub><mi>p</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>≤</mo><msub><mtext>T</mtext><mn>2</mn></msub></mrow></mtd></mtr><mtr><mtd><mrow></mrow></mtd><mtd><mrow><mo>=</mo><mn>2</mn><msub><mrow><mtext> for T</mtext></mrow><mn>2</mn></msub><mo>&lt;</mo><msub><mi>p</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>≤</mo><msub><mtext>T</mtext><mn>3</mn></msub></mrow></mtd></mtr><mtr><mtd><mrow></mrow></mtd><mtd><mrow><mo>=</mo><mn>3</mn><msub><mrow><mtext> for T</mtext></mrow><mn>3</mn></msub><mo>&lt;</mo><msub><mi>p</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>≤</mo><msub><mtext>T</mtext><mn>4</mn></msub></mrow></mtd></mtr><mtr><mtd><mrow></mrow></mtd><mtd><mrow><mo>=</mo><msub><mrow><mtext>k for T</mtext></mrow><mtext>k</mtext></msub><mo>&lt;</mo><msub><mi>p</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>≤</mo><msub><mtext>T</mtext><mtext>k</mtext></msub></mrow></mtd></mtr><mtr><mtd><mrow></mrow></mtd><mtd><mrow><mo>=</mo><mn>0</mn><mtext> otherwise</mtext></mrow></mtd></mtr></mtable></mrow></math></script></div><div class="l">(2)</div></div><p></p><p id="__p78">Here, the K<sup>th</sup> band is corresponding to object/region having pixel values in the range of T<sub>k</sub> to T<sub>k+1</sub> where T<sub>k</sub> is the lower limit of gray level and T<sub>k+1</sub> is the upper limit of Gray level band.</p><p id="__p79">For application of thresholding based segmentation technique, it is required to apply the correct threshold values in order to achieve proper segmentation results, otherwise results are poor. The histogram of an image is particularly used to determine the value of threshold. The histogram of abdomen CT image is shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0003/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0003" rid-ob="ob-F0003" co-legend-rid="lgnd_F0003"><span>Figure 3</span></a>. There are three peaks (maxima) separated by two minima. The values of these minima are selected as threshold for segmentation of image; the original Abdomen CT image and corresponding segmentation result are shown in figures <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0004" rid-ob="ob-F0004" co-legend-rid="lgnd_F0004"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3.5em;">figures4a</span></span><span>4a</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0005/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0005" rid-ob="ob-F0005" co-legend-rid="lgnd_F0005"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and4b</span></span><span>4b</span></a> respectively.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F0003" co-legend-rid="lgnd_F0003"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0003/" target="figure" rid-figpopup="F0003" rid-ob="ob-F0003"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140507408916032" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g003.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g003.jpg" src="./Automated medical image segmentation techniques_files/JMP-35-3-g003.jpg"></div></a><div id="largeobj_idm140507408916032" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0003/" target="figure" rid-figpopup="F0003" rid-ob="ob-F0003"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0003/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F0003"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0003/" target="figure" rid-figpopup="F0003" rid-ob="ob-F0003">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p80">Image Histogram (three peaks separated by two minima)</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F0004" co-legend-rid="lgnd_F0004"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0004/" target="figure" rid-figpopup="F0004" rid-ob="ob-F0004"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140507414573280" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g004.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g004.jpg" src="./Automated medical image segmentation techniques_files/JMP-35-3-g004.jpg"></div></a><div id="largeobj_idm140507414573280" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0004/" target="figure" rid-figpopup="F0004" rid-ob="ob-F0004"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0004/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F0004"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0004/" target="figure" rid-figpopup="F0004" rid-ob="ob-F0004">Figure 4a</a></div><!--caption a7--><div class="caption"><p id="__p81">Original Abdomen CT Image</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F0005" co-legend-rid="lgnd_F0005"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0005/" target="figure" rid-figpopup="F0005" rid-ob="ob-F0005"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140507412595600" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g005.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g005.jpg" src="./Automated medical image segmentation techniques_files/JMP-35-3-g005.jpg"></div></a><div id="largeobj_idm140507412595600" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0005/" target="figure" rid-figpopup="F0005" rid-ob="ob-F0005"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0005/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F0005"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0005/" target="figure" rid-figpopup="F0005" rid-ob="ob-F0005">Figure 4b</a></div><!--caption a7--><div class="caption"><p id="__p82">Segmentation of Abdomen (CT image using threshold technique)</p></div></div></div><p id="__p83">Object 1 belongs (0 to 55)</p><p id="__p84">Object 2 belongs (55 to 200)</p><p id="__p85">Object 3 belongs (200 to 255)</p><p id="__p86" class="p p-last">For histogram-based optimal segmentation of images numbers of methods have been proposed by different researchers.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT11" rid="CIT11" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">11</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT15" rid="CIT15" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">15</a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT17" rid="CIT17" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">17</a>] Frank <em>et al</em>,[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT17" rid="CIT17" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">17</a>] have used optimal threshold selection method for segmentation of T1 weighted MR brain image to segment gray matter, white matter, and cerebro spinal fluid.</p></div><div id="sec2-14" class="sec"><h3>Limitations</h3><p id="__p87" class="p p-first-last">Selection of proper values of threshold is quite difficult. Performance is affected in presence of artifacts.</p></div><div id="sec2-15" class="sec"><h3>Edge based segmentation</h3><p id="__p88" class="p p-first">Edge based segmentation is the most common method based on detection of edges i.e. boundaries which separate distinct regions. Edge detection method is based on marking of discontinuities in gray level, color etc., and often these edges represent boundaries between objects. This method divides an image on the basis of boundaries.</p><p id="__p89">Number of edge detecting operators based on gradient (derivative) function are available e.g. Prewitt, Sobel, Roberts (1<sup>st</sup> derivative type) and Laplacian (2<sup>nd</sup> derivative type), Canny, Marr-Hilclrath edge detector. Further, in edge based segmentation method, it is required to build the border by combining the detected edges into a edge chain in this process the spurious, or fake edges, weak edges are removed by thresholding operation. The different edge based segmentation algorithms are:</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>Edge relaxation,[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT18" rid="CIT18" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">18</a>]</div></li><li><div>Border detection method,[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT19" rid="CIT19" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">19</a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT21" rid="CIT21" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">21</a>]</div></li><li><div>Hough transform based[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT22" rid="CIT22" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">22</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT23" rid="CIT23" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">23</a>]</div></li></ul><p id="__p93">The generalized algorithm for edge based segmentation has the following steps.</p><ol class="enumerated" style="list-style-type:decimal"><!--
list-behavior=enumerated
prefix-word=
mark-type=decimal
max-label-size=0
--><li><div>Apply the derivative operator to detect edges of the image</div></li><li><div>Measure the strength of edges by measuring amplitude of the gradient</div></li><li><div>Retain all edge having magnitude greater than threshold T (removal of weak edge)</div></li><li><div>Find the position of crack edges; the crack edge is either retained or rejected based on the confidence it receives from it predecessor and successor edges</div></li><li><div>Step 3 and 4 are repeated with different values of threshold so as to find out the closed boundaries; segmentation of an image is achieved</div></li></ol><p id="__p99"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0006/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0006" rid-ob="ob-F0006" co-legend-rid="lgnd_F0006"><span>Figure 5</span></a> shows the result of edge based segmentation of abdomen CT image [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0004" rid-ob="ob-F0004" co-legend-rid="lgnd_F0004"><span>Figure 4a</span></a>], in the present result canny edge detector has been employed.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F0006" co-legend-rid="lgnd_F0006"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0006/" target="figure" rid-figpopup="F0006" rid-ob="ob-F0006"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140507413874992" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g006.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g006.jpg" src="./Automated medical image segmentation techniques_files/JMP-35-3-g006.jpg"></div></a><div id="largeobj_idm140507413874992" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0006/" target="figure" rid-figpopup="F0006" rid-ob="ob-F0006"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0006/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F0006"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0006/" target="figure" rid-figpopup="F0006" rid-ob="ob-F0006">Figure 5</a></div><!--caption a7--><div class="caption"><p id="__p100">Result of Edge-based Segmentation of Abdomen (CT image)</p></div></div></div><p id="__p101" class="p">The limitations of edge based method are:</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>Performance is affected by the presence of noise</div></li><li><div>fake edges and weak edges may be present in the detected edge image which may have a negative influence on segmentation results</div></li><li><div>Edge detection techniques are required to be used in conjunction with region-based technique for complete segmentation.</div></li></ul></div><div id="sec2-16" class="sec"><h3>Region based segmentation</h3><p id="__p105" class="p p-first">Region based methods are based on the principle of homogeneity - pixels with similar properties are clustered together to form a homogenous region. The criteria for homogeneity is most of the time gray level of pixels[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT24" rid="CIT24" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">24</a>] and this criteria can be specified by following conditions</p><span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-3-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M3" overflow="scroll" role="math" style="width: 14.468em; display: inline-block;"><span style="display: inline-block; position: relative; width: 11.646em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(1.313em, 1011.65em, 3.027em, -999.997em); top: -2.165em; left: 0.003em;"><span class="mrow" id="MathJax-Span-151"><span class="mrow" id="MathJax-Span-152"><span class="msub" id="MathJax-Span-153"><span style="display: inline-block; position: relative; width: 1.212em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.76em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-154" style="font-family: MathJax_Math-italic;">R</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.759em;"><span class="mn" id="MathJax-Span-155" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="munder" id="MathJax-Span-156" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 0.658em; height: 0px;"><span style="position: absolute; clip: rect(3.228em, 1000.61em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mo" id="MathJax-Span-157" style="font-family: MathJax_Main;">∪</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.732em, 1000.15em, 4.236em, -999.997em); top: -3.374em; left: 0.255em;"><span class="mo" id="MathJax-Span-158" style="font-size: 70.7%; font-family: MathJax_Main;">.</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msub" id="MathJax-Span-159" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.212em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.76em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-160" style="font-family: MathJax_Math-italic;">R</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.759em;"><span class="mn" id="MathJax-Span-161" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="munder" id="MathJax-Span-162" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 0.658em; height: 0px;"><span style="position: absolute; clip: rect(3.228em, 1000.61em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mo" id="MathJax-Span-163" style="font-family: MathJax_Main;">∪</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.732em, 1000.15em, 4.236em, -999.997em); top: -3.374em; left: 0.255em;"><span class="mo" id="MathJax-Span-164" style="font-size: 70.7%; font-family: MathJax_Main;">.</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msub" id="MathJax-Span-165" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.212em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.76em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-166" style="font-family: MathJax_Math-italic;">R</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.759em;"><span class="mn" id="MathJax-Span-167" style="font-size: 70.7%; font-family: MathJax_Main;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="munder" id="MathJax-Span-168" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 0.658em; height: 0px;"><span style="position: absolute; clip: rect(3.228em, 1000.61em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mo" id="MathJax-Span-169" style="font-family: MathJax_Main;">∪</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.732em, 1000.15em, 4.236em, -999.997em); top: -3.374em; left: 0.255em;"><span class="mo" id="MathJax-Span-170" style="font-size: 70.7%; font-family: MathJax_Main;">.</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mn" id="MathJax-Span-171" style="font-family: MathJax_Main; padding-left: 0.204em;">...</span><span class="munder" id="MathJax-Span-172" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 0.658em; height: 0px;"><span style="position: absolute; clip: rect(3.228em, 1000.61em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mo" id="MathJax-Span-173" style="font-family: MathJax_Main;">∪</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.732em, 1000.15em, 4.236em, -999.997em); top: -3.374em; left: 0.255em;"><span class="mo" id="MathJax-Span-174" style="font-size: 70.7%; font-family: MathJax_Main;">.</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msub" id="MathJax-Span-175" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.061em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.76em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-176" style="font-family: MathJax_Math-italic;">R</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.759em;"><span class="mi" id="MathJax-Span-177" style="font-size: 70.7%; font-family: MathJax_Math-italic;">i</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-178" style="font-family: MathJax_Main; padding-left: 0.305em;">=</span><span class="mi" id="MathJax-Span-179" style="font-family: MathJax_Math-italic; padding-left: 0.305em;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.17em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.934em; border-left: 0px solid; width: 0px; height: 1.878em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" overflow="scroll"><mrow><msub><mi>R</mi><mn>1</mn></msub><munder><mo>∪</mo><mo>.</mo></munder><msub><mi>R</mi><mn>2</mn></msub><munder><mo>∪</mo><mo>.</mo></munder><msub><mi>R</mi><mn>3</mn></msub><munder><mo>∪</mo><mo>.</mo></munder><mn>...</mn><munder><mo>∪</mo><mo>.</mo></munder><msub><mi>R</mi><mi>i</mi></msub><mo>=</mo><mi>I</mi></mrow></math></script><p id="__p106">where <em>R<sub>1</sub>, R<sub>2</sub>, R<sub>3</sub>, …R<sub>i</sub></em> are the region in the image <em>I</em>,</p><p id="__p107">and further, <span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-4-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M4" overflow="scroll" role="math" style="width: 14.821em; display: inline-block;"><span style="display: inline-block; position: relative; width: 11.948em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(0.557em, 1011.75em, 3.279em, -999.997em); top: -2.165em; left: 0.003em;"><span class="mrow" id="MathJax-Span-181"><span class="mtable" id="MathJax-Span-182" style="padding-right: 0.154em; padding-left: 0.154em;"><span style="display: inline-block; position: relative; width: 11.646em; height: 0px;"><span style="position: absolute; clip: rect(2.472em, 1011.65em, 4.892em, -999.997em); top: -3.979em; left: 0.003em;"><span style="display: inline-block; position: relative; width: 11.646em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1011.65em, 4.287em, -999.997em); top: -4.635em; left: 0.003em;"><span class="mtd" id="MathJax-Span-183"><span class="mrow" id="MathJax-Span-184"><span class="msub" id="MathJax-Span-185"><span style="display: inline-block; position: relative; width: 1.212em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.76em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-186" style="font-family: MathJax_Math-italic;">R</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.759em;"><span class="mn" id="MathJax-Span-187" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-188" style="font-family: MathJax_Main; padding-left: 0.204em;">∩</span><span class="msub" id="MathJax-Span-189" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.212em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.76em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-190" style="font-family: MathJax_Math-italic;">R</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.759em;"><span class="mn" id="MathJax-Span-191" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-192" style="font-family: MathJax_Main; padding-left: 0.204em;">∩</span><span class="msub" id="MathJax-Span-193" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.212em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.76em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-194" style="font-family: MathJax_Math-italic;">R</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.759em;"><span class="mn" id="MathJax-Span-195" style="font-size: 70.7%; font-family: MathJax_Main;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-196" style="font-family: MathJax_Main; padding-left: 0.204em;">∩</span><span class="mn" id="MathJax-Span-197" style="font-family: MathJax_Main; padding-left: 0.204em;">...</span><span class="mo" id="MathJax-Span-198" style="font-family: MathJax_Main; padding-left: 0.204em;">∩</span><span class="msub" id="MathJax-Span-199" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.061em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.76em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-200" style="font-family: MathJax_Math-italic;">R</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.759em;"><span class="mi" id="MathJax-Span-201" style="font-size: 70.7%; font-family: MathJax_Math-italic;">i</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-202" style="font-family: MathJax_Main; padding-left: 0.305em;">=</span><span class="mn" id="MathJax-Span-203" style="font-family: MathJax_Main; padding-left: 0.305em;">0</span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.833em, 1000em, 4.136em, -999.997em); top: -3.223em; left: 0.003em;"><span class="mtd" id="MathJax-Span-204"><span class="mrow" id="MathJax-Span-205"><span class="mrow" id="MathJax-Span-206"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.17em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.247em; border-left: 0px solid; width: 0px; height: 3.128em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-4"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M4" overflow="scroll"><mtable columnalign="left"><mtr><mtd><msub><mi>R</mi><mn>1</mn></msub><mo>∩</mo><msub><mi>R</mi><mn>2</mn></msub><mo>∩</mo><msub><mi>R</mi><mn>3</mn></msub><mo>∩</mo><mn>...</mn><mo>∩</mo><msub><mi>R</mi><mi>i</mi></msub><mo>=</mo><mn>0</mn></mtd></mtr><mtr><mtd><mrow></mrow></mtd></mtr></mtable></math></script></p><p id="__p108">This is as per the set theory of homogeneity.</p><p id="__p109" class="p">Region based segmentation is further divided into three types based on the principle of region growing:</p><ol class="enumerated" style="list-style-type:lower-alpha"><!--
list-behavior=enumerated
prefix-word=
mark-type=lower-alpha
max-label-size=0
--><li><div>Region merging</div></li><li><div>Region splitting</div></li><li><div>Split and merge</div></li></ol></div><div id="sec2-17" class="sec"><h3>Region merging</h3><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>In this method some seeding points are required to initialize the process, the segmentation results are dependent on the choice of seeds.</div></li><li><div>Regions are grown iteratively by merging the neighboring pixels depending upon the merging criterion.</div></li><li><div>This process is continued until all pixels are assigned to their respective regions as per merging criterion.</div></li></ul></div><div id="sec2-18" class="sec"><h3>Region splitting:</h3><p id="__p116" class="p p-first-last">Its principle is just opposite to region merging and whole image is continuously split until no further splitting of a region is possible.</p></div><div id="sec2-19" class="sec"><h3>Split and merge method:</h3><p id="__p117" class="p p-first">This is the combination of splits and merges utilizing the advantage of the two methods. This method is based on quad quadrant tree representation of data whereby image segment is split into four quadrants provided the original segment is non-uniform in properties. After this the four neighboring squares are merged depending on the uniformity of the region (segments). This split and merge process is continued until no further split and merge is possible.</p><p id="__p118">The algorithm for split and merge follows the following steps.</p><ol class="enumerated" style="list-style-type:decimal"><!--
list-behavior=enumerated
prefix-word=
mark-type=decimal
max-label-size=0
--><li><div>Define homogeneity criterion. Break image into four square quadrants</div></li><li><div>If any resultant square is not homogeneous split it further into four quadrants</div></li><li><div>At each level merge the two or more neighboring regions satisfying the condition of homogeneity</div></li><li><div>Continue the split and merge until no further split and merge of region is possible</div></li></ol><p id="__p123">Apart from the above-mentioned techniques watershed segmentation based on the concept of topography and hydrography is also a region-based segmentation.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT25" rid="CIT25" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">25</a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT28" rid="CIT28" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">28</a>]</p><p id="__p124"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0007/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0007" rid-ob="ob-F0007" co-legend-rid="lgnd_F0007"><span>Figure 6</span></a> shows the result of region based segmentation of abdomen CT image [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0004/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0004" rid-ob="ob-F0004" co-legend-rid="lgnd_F0004"><span>Figure 4a</span></a>]. K-mean clustering algorithm has been used in present case to cluster the pixels having similar gray levels.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F0007" co-legend-rid="lgnd_F0007"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0007/" target="figure" rid-figpopup="F0007" rid-ob="ob-F0007"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140507411852768" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g007.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g007.jpg" src="./Automated medical image segmentation techniques_files/JMP-35-3-g007.jpg"></div></a><div id="largeobj_idm140507411852768" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0007/" target="figure" rid-figpopup="F0007" rid-ob="ob-F0007"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0007/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F0007"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0007/" target="figure" rid-figpopup="F0007" rid-ob="ob-F0007">Figure 6</a></div><!--caption a7--><div class="caption"><p id="__p125">Segmentation of Abdomen (CT image using region based technique)</p></div></div></div><p id="__p126" class="p">The limitation of region based segmentation is that there are chances of under segmentation and over segmentation of regions in the image. However, this problem can be rectified in two ways</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>By optimally selecting the criterion for segmentation, for this several algorithm utilizing artificial intelligence techniques have been developed.</div></li><li><div>By combining region based approach with edge based approach.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT12" rid="CIT12" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">12</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT29" rid="CIT29" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">29</a>]</div></li></ul></div><div id="sec2-20" class="sec"><h3>Method based on the textural features</h3><p id="__p129" class="p p-first">Textural features of image are important from image segmentation and classification point of view. Different researchers have used these features to achieve image segmentation, classification, and both segmentation as well as classification. The aim of texture based segmentation method is to subdivide the image into region having different texture properties, while in classification the aim is to classify the regions which have already been segmented by one or other method.</p><div id="sec3-7" class="sec sec-last"><p></p><h4 class="inline">Definition of texture: </h4><p id="__p130" class="p p-first">Texture is defined as something consisting of mutually related elements.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT24" rid="CIT24" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">24</a>] A texture may be fine coarse, smooth, or grained depending upon its tone and structure. While tone is based on pixel intensity properties, structure is the spatial relationship between pixels.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT30" rid="CIT30" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">30</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT31" rid="CIT31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966062" role="button" aria-expanded="false" aria-haspopup="true">31</a>] Further texture can be defined as the spatial arrangements of texture primitives or texture elements (also called as textone), arranged in more or less periodic manner. Texture primitive is a group of pixels representing the simplest or basic sub pattern and follows three main approaches for texture feature extraction based on the type of approach used.</p><ol class="enumerated" style="list-style-type:decimal"><!--
list-behavior=enumerated
prefix-word=
mark-type=decimal
max-label-size=0
--><li><div>Statistical approach</div></li><li><div>Syntactic or structural approach and</div></li><li><div>Spectral approach</div></li></ol><p id="__p134">In case of statistical approach, texture is defined by a set of statistically extracted features represented as vector in multidimensional feature space. The statistical features could be based on first-order, second-order or higher-order statistics of gray level of an image. The feature vector so generated from patterns is assigned to their specific class by probabilistic or deterministic decision algorithm.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT32" rid="CIT32" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">32</a>] In case of syntactic approach, texture is defined by texture primitives which are spatially organized according to placement rules to generate complete pattern.</p><p id="__p135">In syntactic feature based pattern recognition, a formal analogy is drawn between the structural pattern and the syntax of language.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT33" rid="CIT33" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">33</a>]</p><p id="__p136">In spectral method, textures are defined by spatial frequencies and evaluated by autocorrelation function of a texture.</p><p id="__p137">Some methods available for textural feature extraction and classification based on the above approaches are: co-occurrence matrix method based on statistical description of gray level of an image,[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT34" rid="CIT34" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">34</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT35" rid="CIT35" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">35</a>] gray level run length method,[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT36" rid="CIT36" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">36</a>] fractal texture description method,[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT37" rid="CIT37" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">37</a>] syntactic method[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT38" rid="CIT38" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">38</a>] and Fourier filter method.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT39" rid="CIT39" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">39</a>]</p><p id="__p138">Comparing the above-mentioned three approaches; spectral frequency-based methods are less efficient while statistical methods are particularly useful for random patterns/textures and for complex patterns, syntactic or structural methods give better results.</p><p id="__p139" class="p p-last">Texture based methods as best suited for segmentation of medical image, when compared to segmentation of medical image using simple gray level based methods.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT40" rid="CIT40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577490" role="button" aria-expanded="false" aria-haspopup="true">40</a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT42" rid="CIT42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577486" role="button" aria-expanded="false" aria-haspopup="true">42</a>]</p></div></div><div id="sec2-21" class="sec"><h3>Other approaches of segmentation</h3><p id="__p140" class="p p-first">Apart from the above methods, the following two methods of image segmentation are also available.</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>Model based segmentation and</div></li><li><div>Atlas based segmentation.</div></li></ul></div><div id="sec2-22" class="sec"><h3>Model based segmentation:</h3><p id="__p143" class="p p-first">The basic approach is that the structure of organs has a repetitive form of geometry and can be modeled probabilistically for variation of shape and geometry. This can be used as constraint while segmenting the image and involves:</p><ol class="enumerated" style="list-style-type:decimal"><!--
list-behavior=enumerated
prefix-word=
mark-type=decimal
max-label-size=0
--><li><div>registration of the training data.</div></li><li><div>probabilistic representation of variation of registered data.</div></li><li><div>statistical influence between model and image.</div></li></ol><p id="__p147" class="p p-last">Model based methods of segmentation involve active shape and appearance model, deformable models and level-set based models.</p></div><div id="sec2-23" class="sec"><h3>Disadvantages</h3><ol class="enumerated" style="list-style-type:decimal"><!--
list-behavior=enumerated
prefix-word=
mark-type=decimal
max-label-size=0
--><li><div>They require manual interaction to place an initial model and choose appropriate parameters.</div></li><li><div>Standard deformable models can also exhibit poor convergence to concave boundaries.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT10" rid="CIT10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577492" role="button" aria-expanded="false" aria-haspopup="true">10</a>]</div></li></ol><p id="__p150" class="p p-last"><strong>Atlas based segmentation approaches</strong> are the most frequently used and powerful approaches in the field of medical image segmentation. In this, information on anatomy, shape, size, and features of different, organs, soft tissues is compiled in the form of atlas or look up table (LUT). Atlas guided approaches are similar to co-relation approaches and the plus point of atlas based approaches is - it performs segmentation and classification in one go. Atlas based segmentation approaches are among the third-generation algorithms. There are indications that certain atlas based methods can compete with manual segmentations although atlas selection, atlas registration procedure, and the manual tracing protocol used in atlas formation are factors that can affect performance.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT1" rid="CIT1" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">1</a>] However, they face limitations in segmenting complex structure with variable shape, size, and properties and expert knowledge is required in building the database.</p></div><div id="sec2-24" class="sec"><h3>Artificial Intelligence Tools for Segmentation and Classification</h3><p id="__p151" class="p p-first-last"><strong>Automatic segmentation</strong> methods have been based on artificial intelligence (AI) based techniques. AI techniques can be classified as supervised and unsupervised. Supervised segmentation requires operator interaction throughout the segmentation process whereas unsupervised methods generally require operator involvement only after segmentation is complete. Unsupervised methods are preferred to ensure a reproducible result[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT43" rid="CIT43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966059" role="button" aria-expanded="false" aria-haspopup="true">43</a>]; however, operator interaction is still required for error correction in the event of an inadequate result.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT44" rid="CIT44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966057" role="button" aria-expanded="false" aria-haspopup="true">44</a>]</p></div><div id="sec2-25" class="sec"><h3>Supervised methods</h3><p id="__p152" class="p p-first">In the supervised category, we can place mostly Artificial Neural Network (ANN) based algorithms. ANN is composed of large number of interconnected processing elements (artificial neurons) working in unison to solve specific problems. The main advantages of ANN are:</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>ability to learn adaptively, using training data to solve complex problems.</div></li><li><div>capability of self-organization; it can create its own organization depending upon the information it receives during learning time</div></li><li><div>capability of performance in real time because of parallel configuration</div></li></ul><p id="__p156" class="p p-last">In case of ANN, learning is achieved by the adaptation of weights and bias of the neurons with respect to the training procedure and training data. ANN has been widely used for segmentation and classification purposes in both supervised and unsupervised modes.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT45" rid="CIT45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577491" role="button" aria-expanded="false" aria-haspopup="true">45</a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT47" rid="CIT47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577493" role="button" aria-expanded="false" aria-haspopup="true">47</a>] Although a variety of different neural network based algorithms have been developed for texture based segmentation and classification with good classification accuracy,[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT47" rid="CIT47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577489" role="button" aria-expanded="false" aria-haspopup="true">47</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT48" rid="CIT48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966067" role="button" aria-expanded="false" aria-haspopup="true">48</a>] most of these texture classifier algorithms require extensive supervision, training; their performance is sensitive to training parameters and is adversely affected in the presence of noise. At times supervised image segmentation and classification methods become very expensive, difficult and even impossible to correctly select and label the training data with its true category.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT49" rid="CIT49" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">49</a>] Training is the main requirement of many ANN based algorithms where the classifiers need to be trained before it can be applied to segmentation and classification problem. Further, for different data sets, analysis of different images of different type and format, the whole effort of selecting training data set and training is required to be redone.</p></div><div id="sec2-26" class="sec"><h3>Unsupervised methods</h3><p id="__p157" class="p p-first">Most of the unsupervised algorithms are cluster based and not dependent on training and training data. The two commonly used algorithms for clustering are K-mean or Hard C-mean and Fuzzy C-means.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT50" rid="CIT50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966065" role="button" aria-expanded="false" aria-haspopup="true">50</a>] K-means algorithm produces results that correspond to hard segmentation while fuzzy C-mean produces soft segmentation which can be converted into hard segmentation by allowing the pixels to have membership of cluster in which they have maximum value of membership coefficients.</p><p id="__p158">In clustering, the aim is to construct decision boundaries based on unlabeled training data.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT49" rid="CIT49" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">49</a>] Clustering is the process of finding natural grouping clusters in multidimensional feature space. It is difficult because clusters of different shapes and sizes can occur in multidimensional feature space. A number of functional definitions of clusters have been proposed: Patterns within a cluster are more similar to each other than patterns belonging to different clusters.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT49" rid="CIT49" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">49</a>] Image segmentation may be considered a clustering[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT51" rid="CIT51" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">51</a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT53" rid="CIT53" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">53</a>] process in which the pixels are classified into the attribute regions based on the texture feature vector calculated around the pixel local neighborhood. Fuzzy clustering is a good method of classifying collection of data point to reside in multiple clusters with different degrees of membership (fuzzy c mean algorithm).[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT50" rid="CIT50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966064" role="button" aria-expanded="false" aria-haspopup="true">50</a>]</p><p id="__p159" class="p p-last">However, the main limitations of fuzzy clustering algorithm are: (a) sensitivity to initial partition matrix (b) stopping criterion (c) solution may get stuck at local minima. Hence, clustering techniques may not result in optimal solution and there is no best clustering algorithm for a particular application. A number of different algorithms are required to be tried to find the best one.</p></div><div id="sec2-27" class="sec"><h3>Segmentation of CT and MR Images</h3><p id="__p160" class="p p-first">Segmentation of CT and MR images involves three main image related problems; <em>noise</em> that can alter the intensity of a pixel such that its classification becomes uncertain, <em>intensity inhomogeneity</em> where the intensity level of a single tissue class varies gradually over the extent of the image, and images have finite pixel size and are subject to <em>partial volume averaging</em> where individual pixel volumes contain a mixture of tissue classes so that the intensity of a pixel in the image may not be consistent with any one class.</p><p id="__p161" class="p">Some methods available for CT image segmentation are:</p><ul class="unordered" style="list-style-type:disc"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><div>threshold based[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT54" rid="CIT54" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">54</a>]</div></li><li><div>region based[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT55" rid="CIT55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577488" role="button" aria-expanded="false" aria-haspopup="true">55</a>]</div></li><li><div>deformable models based[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT56" rid="CIT56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577484" role="button" aria-expanded="false" aria-haspopup="true">56</a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT57" rid="CIT57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966069" role="button" aria-expanded="false" aria-haspopup="true">57</a>]</div></li><li><div>fuzzy based[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT58" rid="CIT58" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">58</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT59" rid="CIT59" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">59</a>]</div></li><li><div>neural network based[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT45" rid="CIT45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577485" role="button" aria-expanded="false" aria-haspopup="true">45</a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT47" rid="CIT47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577495" role="button" aria-expanded="false" aria-haspopup="true">47</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT60" rid="CIT60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577497" role="button" aria-expanded="false" aria-haspopup="true">60</a>]</div></li></ul></div><div id="sec2-28" class="sec"><h3>Methods available for MR image segmentation:</h3><p id="__p167" class="p p-first">MR imaging is specifically used in brain imaging and thus lot of research work has been done particularly in the areas of MR brain image segmentation.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT61" rid="CIT61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966060" role="button" aria-expanded="false" aria-haspopup="true">61</a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT64" rid="CIT64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_483114800" role="button" aria-expanded="false" aria-haspopup="true">64</a>] The main goal in brain MR segmentation is to segment gray matter, white matter and cerebrospinal fluid. Segmentation is also used to find out the regions corresponding to lesions tumors, cyst, edema, and other pathologies and for this mostly T1- weighted images are used.</p><p id="__p168" class="p p-last">Most of the segmentation methods available for CT and MR images segmentation are intensity based i.e. gray level based hence; the segmentation results are affected by (1) intensity in-homogeneities and (2) partial volume effects. Accordingly, different researchers have proposed methods for correction of these problems.</p></div><div id="sec2-29" class="sec"><h3>Intensity In-homogeneity Correction:</h3><p id="__p169" class="p p-first-last">In MRI intensity, inhomogeneity artifacts cause shading effect to appear over the images.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT65" rid="CIT65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577483" role="button" aria-expanded="false" aria-haspopup="true">65</a>] As a result, they affect the segmentation result while using simple gray level based segmentation techniques. One way to rectify these intensity inhomogeneity artifacts is by using image enhancement techniques.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT66" rid="CIT66" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">66</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT67" rid="CIT67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577487" role="button" aria-expanded="false" aria-haspopup="true">67</a>] Further, for this segmentation, techniques based on statistical methods[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT68" rid="CIT68" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966068" role="button" aria-expanded="false" aria-haspopup="true">68</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT69" rid="CIT69" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966061" role="button" aria-expanded="false" aria-haspopup="true">69</a>] and fuzzy methods,[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT58" rid="CIT58" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">58</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT70" rid="CIT70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966058" role="button" aria-expanded="false" aria-haspopup="true">70</a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT73" rid="CIT73" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_310577498" role="button" aria-expanded="false" aria-haspopup="true">73</a>] which gives soft segmentation results are particularly useful in overcoming the above mentioned limitation. A review of methods for correction of intensity inhomogeneity artifact in MR images has been presented by Vovk <em>et al</em>.[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT74" rid="CIT74" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222966063" role="button" aria-expanded="false" aria-haspopup="true">74</a>]</p></div><div id="sec2-30" class="sec sec-last"><h3>Partial Volume Effect Correction:</h3><p id="__p170" class="p p-first">When multiple tissues contribute to single pixel or voxel the resultant image is blurred at boundaries of the different region or object and this effect is called as partial volume effect. To deal with partial volume effect, soft segmentation is a good option. In soft segmentation we allow the region or classes to overlap i.e. pixels are allowed to have multiple memberships with varying degree of membership coefficient in different regions. In hard segmentation, we do not allow overlapping of the segmented region and the pixels are forced to reside in the region in which they are having maximum membership. Thus soft segmentation retains more information about the original image by allowing the pixel to have membership in the multiple regions.</p><p id="__p171">For hard segmentation the pixel is having binary membership defined as</p><span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-5-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M5" overflow="scroll" role="math" style="width: 12.251em; display: inline-block;"><span style="display: inline-block; position: relative; width: 9.882em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(1.767em, 1009.68em, 4.589em, -999.997em); top: -3.425em; left: 0.003em;"><span class="mrow" id="MathJax-Span-208"><span class="mrow" id="MathJax-Span-209"><span class="msub" id="MathJax-Span-210"><span style="display: inline-block; position: relative; width: 1.817em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.86em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-211" style="font-family: MathJax_Math-italic;">m</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.859em;"><span class="mrow" id="MathJax-Span-212"><span class="mi" id="MathJax-Span-213" style="font-size: 70.7%; font-family: MathJax_Math-italic;">k</span><span class="mo" id="MathJax-Span-214" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-215" style="font-size: 70.7%; font-family: MathJax_Math-italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-216" style="font-family: MathJax_Main; padding-left: 0.305em;">=</span><span class="mrow" id="MathJax-Span-217" style="padding-left: 0.305em;"><span class="mo" id="MathJax-Span-218" style="vertical-align: 0.003em;"><span style="font-family: MathJax_Size3;">{</span></span><span class="mrow" id="MathJax-Span-219"><span class="mtable" id="MathJax-Span-220" style="padding-right: 0.154em; padding-left: 0.154em;"><span style="display: inline-block; position: relative; width: 5.648em; height: 0px;"><span style="position: absolute; clip: rect(2.472em, 1000.46em, 4.992em, -999.997em); top: -3.979em; left: 0.003em;"><span style="display: inline-block; position: relative; width: 0.507em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.41em, 4.136em, -999.997em); top: -4.685em; left: 50%; margin-left: -0.249em;"><span class="mtd" id="MathJax-Span-221"><span class="mrow" id="MathJax-Span-222"><span class="mn" id="MathJax-Span-223" style="font-family: MathJax_Main;">1</span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.178em, 1000.46em, 4.136em, -999.997em); top: -3.173em; left: 50%; margin-left: -0.249em;"><span class="mtd" id="MathJax-Span-235"><span class="mrow" id="MathJax-Span-236"><span class="mn" id="MathJax-Span-237" style="font-family: MathJax_Main;">0</span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.422em, 1004.29em, 4.942em, -999.997em); top: -3.979em; left: 1.313em;"><span style="display: inline-block; position: relative; width: 4.337em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1003.68em, 4.438em, -999.997em); top: -4.685em; left: 50%; margin-left: -1.862em;"><span class="mtd" id="MathJax-Span-224"><span class="mrow" id="MathJax-Span-225"><span class="mrow" id="MathJax-Span-226"><span class="mi" id="MathJax-Span-227" style="font-family: MathJax_Math-italic;">i</span><span class="msub" id="MathJax-Span-228"><span style="display: inline-block; position: relative; width: 0.859em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.56em, 4.337em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-229" style="font-family: MathJax_Math-italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mi" id="MathJax-Span-230" style="font-size: 70.7%; font-family: MathJax_Math-italic;">j</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-231" style="font-family: MathJax_Main; padding-left: 0.305em;">∈</span><span class="msub" id="MathJax-Span-232" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 1.212em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1000.76em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-233" style="font-family: MathJax_Math-italic;">R</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.759em;"><span class="mi" id="MathJax-Span-234" style="font-size: 70.7%; font-family: MathJax_Math-italic;">k</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.128em, 1004.29em, 4.136em, -999.997em); top: -3.173em; left: 50%; margin-left: -2.165em;"><span class="mtd" id="MathJax-Span-238"><span class="mrow" id="MathJax-Span-239"><span class="mrow" id="MathJax-Span-240"><span class="mi" id="MathJax-Span-241" style="font-family: MathJax_Math-italic;">o</span><span class="mi" id="MathJax-Span-242" style="font-family: MathJax_Math-italic;">t</span><span class="mi" id="MathJax-Span-243" style="font-family: MathJax_Math-italic;">h</span><span class="mi" id="MathJax-Span-244" style="font-family: MathJax_Math-italic;">e</span><span class="mi" id="MathJax-Span-245" style="font-family: MathJax_Math-italic;">r</span><span class="mi" id="MathJax-Span-246" style="font-family: MathJax_Math-italic;">w</span><span class="mi" id="MathJax-Span-247" style="font-family: MathJax_Math-italic;">i</span><span class="mi" id="MathJax-Span-248" style="font-family: MathJax_Math-italic;">s</span><span class="mi" id="MathJax-Span-249" style="font-family: MathJax_Math-italic;">e</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.43em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.309em; border-left: 0px solid; width: 0px; height: 3.253em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-5"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M5" overflow="scroll"><mrow><msub><mi>m</mi><mrow><mi>k</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>=</mo><mrow><mo>{</mo><mrow><mtable><mtr><mtd><mn>1</mn></mtd><mtd><mrow><mi>i</mi><msub><mi>f</mi><mi>j</mi></msub><mo>∈</mo><msub><mi>R</mi><mi>k</mi></msub></mrow></mtd></mtr><mtr><mtd><mn>0</mn></mtd><mtd><mrow><mi>o</mi><mi>t</mi><mi>h</mi><mi>e</mi><mi>r</mi><mi>w</mi><mi>i</mi><mi>s</mi><mi>e</mi></mrow></mtd></mtr></mtable></mrow></mrow></mrow></math></script><p id="__p172">Where <em>j</em> is j<sup>th</sup> pixel belongs to image (I), and <em>m<sub>k,j</sub></em> is the membership function of j<sup>th</sup> pixel in region R<sub>k</sub>. Similarly for soft segmentation the pixel has multiple memberships in different regions and the membership function must satisfy the following constraints:</p><span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-6-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="M6" overflow="scroll" role="math" style="width: 12.704em; display: inline-block;"><span style="display: inline-block; position: relative; width: 10.234em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(-0.451em, 1009.98em, 4.287em, -999.997em); top: -2.165em; left: 0.003em;"><span class="mrow" id="MathJax-Span-251"><span class="mtable" id="MathJax-Span-252" style="padding-right: 0.154em; padding-left: 0.154em;"><span style="display: inline-block; position: relative; width: 9.932em; height: 0px;"><span style="position: absolute; clip: rect(2.876em, 1009.88em, 7.513em, -999.997em); top: -5.391em; left: 0.003em;"><span style="display: inline-block; position: relative; width: 9.932em; height: 0px;"><span style="position: absolute; clip: rect(3.128em, 1009.88em, 4.438em, -999.997em); top: -5.643em; left: 0.003em;"><span class="mtd" id="MathJax-Span-253"><span class="mrow" id="MathJax-Span-254"><span class="mn" id="MathJax-Span-255" style="font-family: MathJax_Main;">0</span><span class="mo" id="MathJax-Span-256" style="font-family: MathJax_Main; padding-left: 0.305em;">≤</span><span class="msub" id="MathJax-Span-257" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 1.817em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.86em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-258" style="font-family: MathJax_Math-italic;">m</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.859em;"><span class="mrow" id="MathJax-Span-259"><span class="mi" id="MathJax-Span-260" style="font-size: 70.7%; font-family: MathJax_Math-italic;">k</span><span class="mo" id="MathJax-Span-261" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-262" style="font-size: 70.7%; font-family: MathJax_Math-italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-263" style="font-family: MathJax_Main; padding-left: 0.305em;">≤</span><span class="mn" id="MathJax-Span-264" style="font-family: MathJax_Main; padding-left: 0.305em;">1</span><span class="mtext" id="MathJax-Span-265" style="font-family: MathJax_Main;">&nbsp;for&nbsp;all&nbsp;k,&nbsp;j</span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.119em, 1008.77em, 5.345em, -999.997em); top: -3.223em; left: 0.003em;"><span class="mtd" id="MathJax-Span-266"><span class="mrow" id="MathJax-Span-267"><span class="mtext" id="MathJax-Span-268" style="font-family: MathJax_Main;">and&nbsp;</span><span class="mstyle" id="MathJax-Span-269" style="padding-left: 0.154em;"><span class="mrow" id="MathJax-Span-270"><span class="munderover" id="MathJax-Span-271"><span style="display: inline-block; position: relative; width: 1.464em; height: 0px;"><span style="position: absolute; clip: rect(2.876em, 1001.41em, 4.589em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mo" id="MathJax-Span-272" style="font-family: MathJax_Size2; vertical-align: 0.003em;">∑</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.329em, 1001.21em, 4.236em, -999.997em); top: -2.87em; left: 0.103em;"><span class="mrow" id="MathJax-Span-273"><span class="mi" id="MathJax-Span-274" style="font-size: 70.7%; font-family: MathJax_Math-italic;">k</span><span class="mo" id="MathJax-Span-275" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-276" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.228em, 1000.61em, 4.136em, -999.997em); top: -5.139em; left: 0.406em;"><span class="mi" id="MathJax-Span-277" style="font-size: 70.7%; font-family: MathJax_Math-italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mrow" id="MathJax-Span-278" style="padding-left: 0.154em;"><span class="msub" id="MathJax-Span-279"><span style="display: inline-block; position: relative; width: 1.817em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.86em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-280" style="font-family: MathJax_Math-italic;">m</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.859em;"><span class="mrow" id="MathJax-Span-281"><span class="mi" id="MathJax-Span-282" style="font-size: 70.7%; font-family: MathJax_Math-italic;">k</span><span class="mo" id="MathJax-Span-283" style="font-size: 70.7%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-284" style="font-size: 70.7%; font-family: MathJax_Math-italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-285" style="font-family: MathJax_Main; padding-left: 0.305em;">=</span><span class="mn" id="MathJax-Span-286" style="font-family: MathJax_Main; padding-left: 0.305em;">1</span><span class="mtext" id="MathJax-Span-287" style="font-family: MathJax_Main;">&nbsp;&nbsp;</span><span class="mo" id="MathJax-Span-288" style="font-family: MathJax_Main;">∀</span><span class="mi" id="MathJax-Span-289" style="font-family: MathJax_Math-italic;">j</span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 5.396em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.17em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -2.497em; border-left: 0px solid; width: 0px; height: 5.691em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-6"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M6" overflow="scroll"><mtable columnalign="left"><mtr><mtd><mn>0</mn><mo>≤</mo><msub><mi>m</mi><mrow><mi>k</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>≤</mo><mn>1</mn><mtext> for all k, j</mtext></mtd></mtr><mtr><mtd><mtext>and </mtext><mstyle displaystyle="true"><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></munderover><mrow><msub><mi>m</mi><mrow><mi>k</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>=</mo><mn>1</mn><mtext>  </mtext><mo>∀</mo><mi>j</mi></mrow></mstyle></mtd></mtr></mtable></math></script><p id="__p173">where N = total number of segmented regions in the image (I)</p><p id="__p174">From the value of membership of pixel j in R<sub>k</sub> (k<sup>th</sup> region) one can measure how strongly the pixel belongs to the region; more the membership value more strongly it is the member of region R<sub>k</sub>. At the border region the pixel can have different membership in different regions.</p><p id="__p175" class="p">Fuzzy clustering is an excellent method for soft segmentation[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT58" rid="CIT58" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">58</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT72" rid="CIT72" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">72</a>] and the most widely used unsupervised algorithm for segmentation of both CT and MR images. Soft segmentation based membership can be converted to hard segmentation by allowing the pixel to have the value of membership function one (1) corresponding to the region for which it has highest membership value. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0008" rid-ob="ob-F0008" co-legend-rid="lgnd_F0008"><span>Figure 7A</span></a> shows brain CT image; individual segments of this image obtained using simulated annealing based fuzzy-c-means algorithm[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT58" rid="CIT58" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">58</a>] are shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0008" rid-ob="ob-F0008" co-legend-rid="lgnd_F0008"><span>Figure 7B</span></a>–<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0008" rid-ob="ob-F0008" co-legend-rid="lgnd_F0008"><span>7E</span></a> and <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0008/" target="figure" class="fig-table-link figpopup" rid-figpopup="F0008" rid-ob="ob-F0008" co-legend-rid="lgnd_F0008"><span>Figure 7F</span></a> shows segmented image in pseudo color.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F0008" co-legend-rid="lgnd_F0008"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0008/" target="figure" rid-figpopup="F0008" rid-ob="ob-F0008"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140507410499872" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g008.jpg" title="An external file that holds a picture, illustration, etc.
Object name is JMP-35-3-g008.jpg" src="./Automated medical image segmentation techniques_files/JMP-35-3-g008.jpg"></div></a><div id="largeobj_idm140507410499872" class="largeobj-link align_right" style="display: none"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0008/" target="figure" rid-figpopup="F0008" rid-ob="ob-F0008"></a><a target="object" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0008/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F0008"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/figure/F0008/" target="figure" rid-figpopup="F0008" rid-ob="ob-F0008">Figure 7</a></div><!--caption a7--><div class="caption"><p id="__p176">Individual Segments of Brain CT Image (A) Original (B-E) Individual segments (F) Segmented image in Pseudo Color</p></div></div></div></div></div><div id="sec1-4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1-4title">Conclusion</h2><p id="__p177" class="p p-first">Computer-aided segmentation is a key step finding application in computer aided diagnosis, clinical studies, and treatment planning. In recent years a wide variety of approaches have been proposed to segment CT and MR images having their own merits and limitations. The present review provides the basics of segmentation approaches and their respective features.</p><p id="__p178">The approaches for image segmentation discussed in this review can be ranked on the basis of applicability, suitability, performance, and computational cost. Segmentation techniques based on gray level techniques such as thresholding, and region based techniques are the simplest techniques and find limited applications. However, their performance can be improved by integrating them with artificial intelligence techniques. Techniques based on textural features utilizing atlas or look-up-table have excellent results on medical image segmentation. However, they need expert knowledge in building the atlas. The limitation of atlas based technique is that under certain circumstances it becomes difficult to correctly select and label data; has difficulties in segmenting complex structure with variable shape, size, and properties. In such situations it is better to use unsupervised methods such as fuzzy-c-means algorithm.</p><p id="__p179" class="p p-last">A variety of different neural network-based algorithms are also available for texture-based segmentation and classification having good accuracy. However, most of these neural network-based algorithms require extensive supervision and training and their performance depends upon the training method and data used in training. Finally, it is desired from medical image segmentation and classification algorithms that they must have the following features: a) accuracy, b) reliability, c) repeatability, d) robustness and e) least dependency on the operator.</p></div><div id="idm140507479026928" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140507479026928title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140507479026800"></p><p id="__p180" class="p p-first-last"><strong>Source of Support:</strong> Nil</p><p></p><p class="fn sec" id="idm140507479025552"> </p><p id="__p181" class="p p-first-last"><strong>Conflict of Interest:</strong> None declared.</p><p></p></div></div><div id="idm140507478829296" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140507478829296title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CIT1">1. <span class="element-citation">Withey DJ, Koles ZJ. Three generations of medical image segmentation: Methods and available software. <span><span class="ref-journal">Int J Bioelectromag. </span>2007;<span class="ref-vol">9</span>:67–8.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT2">2. <span class="element-citation">Prince JL, Links JM. Medical imaging signals and system. <span class="ref-journal">Pearson Education.</span> 2006. </span></div><div class="ref-cit-blk half_rhythm" id="CIT3">3. <span class="element-citation">Macovski A.  <span class="ref-journal">Medical imaging systems.</span> Prentice-Hall; 1983. </span></div><div class="ref-cit-blk half_rhythm" id="CIT4">4. <span class="element-citation">Popilock R, Sandrasagaren K, Harris L, Kaser KA. CT artifact recognition for the nuclear technologist. <span><span class="ref-journal">J Nucl Med Technol. </span>2008;<span class="ref-vol">36</span>:79–81.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18483138" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT5">5. <span class="element-citation">Li H, Deklerck R, Cuyper BD, Hermanus A, Nyssen E, Cornelis J. Object recognition in brain CT-scans: Knowledge based fusion of data from multiple feature extractors. <span><span class="ref-journal">IEEE T Med Imaging. </span>1995;<span class="ref-vol">14</span>:212–29.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18215825" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT6">6. <span class="element-citation">Pham DL, Prince JL, Dagher AP, Xu C. An automated technique for statistical characterization of brain tissues in magnetic resonance imaging. <span><span class="ref-journal">Int J Patt Rec Art Intel. </span>1997;<span class="ref-vol">11</span>:1189–211.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT7">7. <span class="element-citation">Gonzalez RC, Woods RE.  <span class="ref-journal">Digital image processing.</span> 2nd ed. 2004.  Pearson Education.</span></div><div class="ref-cit-blk half_rhythm" id="CIT8">8. <span class="element-citation">Pratt KW.  <span class="ref-journal">Digital image processing.</span> 3rd ed. Willey; 2001. pp. 551–87.</span></div><div class="ref-cit-blk half_rhythm" id="CIT9">9. <span class="element-citation">Pal NR, Pal SH. A review on image segmentation techniques. <span><span class="ref-journal">Pattern Recog. </span>1993;<span class="ref-vol">26</span>:1277–94.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT10">10. <span class="element-citation">Pham DL, Xu C, Prince JL. Current methods in medical image segmentation. <span><span class="ref-journal">Ann Rev Biomed Engg. </span>2000;<span class="ref-vol">2</span>:315–37.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/11701515" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT11">11. <span class="element-citation">Ramesh N, Yoo JH, Sethi IK. Thresholding based on histogram approximation. <span><span class="ref-journal">IEEE Proc Vision Image Signal Proc. </span>1995;<span class="ref-vol">142</span>:271–9.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT12">12. <span class="element-citation">Sharma N, Ray AK.  <span class="ref-journal">Proc. of Int. Conf. on Mathematical Biology'.</span> Mathematical Biology recent trends by ANAMAYA Publishers; 2006. Computer aided segmentation of medical images based on hybridized approach of edge and region based techniques; pp. 150–5.</span></div><div class="ref-cit-blk half_rhythm" id="CIT13">13. <span class="element-citation">Wang Z, Gierriero A, Sario M. Comparison of several approaches for segmentation of texture images. <span><span class="ref-journal">Pattern Recog Lett. </span>1996;<span class="ref-vol">17</span>:509–21.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT14">14. <span class="element-citation">Sharma N, Ray AK, Sharma S, Shukla KK, Pradhan S, Aggarwal LM. Segmentation and classification of medical images using texture-primitive features: Application of BAM-type artificial neural network. <span><span class="ref-journal">J Med Physics. </span>2008;<span class="ref-vol">33</span>:119–26.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2772042/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/19893702" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT15">15. <span class="element-citation">Lai CC. A novel image segmentation approach based on particle swarm optimization. <span><span class="ref-journal">IEICE Trans Fundamentals. </span>2006;<span class="ref-vol">E89A</span>:324–7.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT16">16. <span class="element-citation">Brink AD. Minimum spatial entropy threshold selection. <span><span class="ref-journal">IEE Proc Vision Image Signal Proc. </span>1995;<span class="ref-vol">142</span>:128–32.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT17">17. <span class="element-citation">Frank RJ, Grabowski TJ, Damasio H.  <span class="ref-journal">In Abstracts, 25th Annual meeting, Society for Neuroscience, Society for Neuroscience.</span> Washington, DC: 1995. Voxelvise percentage tissue segmentation of human brain Magnetic resonance images; p. 694.</span></div><div class="ref-cit-blk half_rhythm" id="CIT18">18. <span class="element-citation">Hancock ER, Kittler J. Edge labeling using dictionary-based relaxation. <span><span class="ref-journal">IEEE Trans PAMI. </span>1990;<span class="ref-vol">12</span>:165–81.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT19">19. <span class="element-citation">Liow YT. A contour tracing algorithm that preserves common boundaries between regions. <span><span class="ref-journal">CVGIP Image Understanding. </span>1991;<span class="ref-vol">53</span>:313–21.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT20">20. <span class="element-citation">Law T, Itoh H, Seki H. Image filtering, edge detection, and edge tracing using fuzzy reasoning. <span><span class="ref-journal">IEEE Trans PAMI. </span>1996;<span class="ref-vol">18</span>:481–91.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT21">21. <span class="element-citation">Ney H. A comparative study of two search strategies for connected word recognition Dynamic programming and heuristic search. <span><span class="ref-journal">IEEE Trans PAMI. </span>1992;<span class="ref-vol">14</span>:586–95.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT22">22. <span class="element-citation">Xu L, Oja E. Randomized Hough transform (RHT): Basic mechanism and algorithms, and computational complexities. <span><span class="ref-journal">CVGIP Image Comput. </span>1993;<span class="ref-vol">57</span>:131–54.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT23">23. <span class="element-citation">Kalvian H, Hirvonen P, Xu L, Oja E. Probabilistic and non-probabilistic Hough transform: Overview and comparisons. <span><span class="ref-journal">Image Comput. </span>1995;<span class="ref-vol">13</span>:239–52.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT24">24. <span class="element-citation">Sonka M, Hlavac V, Boyle R.  <span class="ref-journal">Image processing, analysis and machine vision.</span> Singapore: Thomson Learning; 1999. </span></div><div class="ref-cit-blk half_rhythm" id="CIT25">25. <span class="element-citation">Ng HP, Ong SH, Foong KW, Nowinski WL. An improved watershed algorithm for medical image segmentation. <span class="ref-journal">Proceedings 12th International Conference on Biomedical Engineering.</span> 2005. </span></div><div class="ref-cit-blk half_rhythm" id="CIT26">26. <span class="element-citation">Nguyen HT, Worring M, Boomgaard RV. Watersnakes: Energy-driven watershed segmentation. <span><span class="ref-journal">IEEE Trans Pattern Anal Mach Intellig. </span>2003;<span class="ref-vol">25</span>:330–42.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT27">27. <span class="element-citation">Najman L, Schmitti M. Geodesic saliency of watershed contours and hierarchical segmentation. <span><span class="ref-journal">IEEE Trans PAMI. </span>1996;<span class="ref-vol">18</span>:12.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT28">28. <span class="element-citation">Lopez AM. Evaluation of methods for ridge and valley detection. <span><span class="ref-journal">IEEE Trans PAMI. </span>1999;<span class="ref-vol">21</span>:327.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT29">29. <span class="element-citation">Gevers T, Smeulders AW.  <span class="ref-journal">Comp Vision Pattern Recognition IEEE Computer Society.</span> Los Almitos, CA: 1997. Combining region splitting and edge detection through guided Delaunay image subdivision; pp. 1021–6.</span></div><div class="ref-cit-blk half_rhythm" id="CIT30">30. <span class="element-citation">Haralick RM. Statistical and structural approaches to texture. <span><span class="ref-journal">Proc IEEE. </span>1979;<span class="ref-vol">67</span>:786–804.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT31">31. <span class="element-citation">Julsez B. Textons, the element of texture perception and their interactions. <span><span class="ref-journal">Nature. </span>1981;<span class="ref-vol">290</span>:91–7.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7207603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT32">32. <span class="element-citation">Fukunaga K.  <span class="ref-journal">Introduction to statistical pattern recognition.</span> 2nd ed. Academic Press; 1990. </span></div><div class="ref-cit-blk half_rhythm" id="CIT33">33. <span class="element-citation">Pavilidis T.  Structural description and graph grammar. In: Chang SK, Fu KS, editors. <span class="ref-journal">Pictorial information systems.</span> Springer Verlag Berlin: 1980. pp. 86–103.</span></div><div class="ref-cit-blk half_rhythm" id="CIT34">34. <span class="element-citation">Argenti F, Alparone L, Benelli G. Fast algorithm for texture analysis using co-occurrence matrices. <span><span class="ref-journal">IEE Proc Part F: Radar Signal Proc. </span>1990;<span class="ref-vol">137</span>:443–8.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT35">35. <span class="element-citation">Haralick RM, Shanmugan K, Dinstein I. Textural features for image classification. <span><span class="ref-journal">IEEE Trans Syst Man Cybern. </span>1973;<span class="ref-vol">3</span>:610–22.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT36">36. <span class="element-citation">Chu CC, Aggarwal JK. The integration of image segmentation maps using region and edge information. <span><span class="ref-journal">IEEE Trans Pattern Anal Mach Intellig. </span>1993;<span class="ref-vol">15</span>:1241–52.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT37">37. <span class="element-citation">Chaudhuri BB, Sarkar N. Texture segmentation using fractal dimension. <span><span class="ref-journal">IEEE Trans PAMI. </span>1995;<span class="ref-vol">17</span>:72–7.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT38">38. <span class="element-citation">Lu SY, Fu KS. A syntactic approach to texture analysis. <span><span class="ref-journal">Comput Graphics Image Proc. </span>1978;<span class="ref-vol">7</span>:303–30.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT39">39. <span class="element-citation">Duda RO, Hart PE, Stork DG.  <span class="ref-journal">Pattern classification.</span> Singapore: Wiley; 2001. pp. 350–93.</span></div><div class="ref-cit-blk half_rhythm" id="CIT40">40. <span class="element-citation">Tesar L, Shimizu A, Smutek D, Kobatake H, Nawano S. Medical image analysis of 3D CT images based on extension of Haralick texture features. <span><span class="ref-journal">Comput Med Imaging Graphics. </span>2008;<span class="ref-vol">32</span>:513–20.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18614335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT41">41. <span class="element-citation">Prasad M, Brown MS, Abtin F, Vasunilsahorn S, Goldin JG. Comparison of scleroderma detection techniques in CT images using supervised and unsupervised methods. <span><span class="ref-journal">Int J Biomed Engg Technol. </span>2008;<span class="ref-vol">1</span>:4.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT42">42. <span class="element-citation">Koss JE, Newman FD, Johnson TK, Krich DL. Abdominal organ segmentation using texture transform and Hopfield neural network. <span><span class="ref-journal">IEEE Trans Med Image. </span>1999;<span class="ref-vol">18</span>:640–8.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/10504097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT43">43. <span class="element-citation">Clarke LP, Velthuizen RP, Camacho MA, Heine JJ, Vaidyanathan M, Hall LO, et al.  ‘MRI segmentation: Methods and applications. <span><span class="ref-journal">Magn Reson Imaging. </span>1995;<span class="ref-vol">13</span>:343–68.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7791545" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT44">44. <span class="element-citation">Olabarriaga SD, Smeulders AW. Interaction in the segmentation of medical images: A survey. <span><span class="ref-journal">Med Image Anal. </span>2001;<span class="ref-vol">5</span>:127–42.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/11516707" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT45">45. <span class="element-citation">Kumar CV, Damyanti G, Pant R, Sreedhar CM. Segmentation and grading of brain tumor an apparent diffusion coefficient image using self organizing maps. <span><span class="ref-journal">Comput Med Imaging Graphica. </span>2007;<span class="ref-vol">31</span>:473–84.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17572068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT46">46. <span class="element-citation">Chaplot S, Patnaik LM, Jagannathan N. Classification of magnetic resonance brain images using wavelets as input to support vector machine and neural network. <span><span class="ref-journal">Biomed Signal Proc Control. </span>2006;<span class="ref-vol">1</span>:86–92.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT47">47. <span class="element-citation">Gletsos M, Mougiakakou SG, Matsopoulos GK, Nikita KS, Nikita AS, Kelekis D. A computer-aided diagnostic system to characterize CT focal liver lesions: Design and optimization of a neural network classifier. <span><span class="ref-journal">IEEE Trans Inform Technol Biomed. </span>2003;<span class="ref-vol">7</span>:153–62.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/14518728" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT48">48. <span class="element-citation">Engeland SV, Timp S, Karssemeijer N. Finding corresponding regions of interest in mediolateral oblique and craniocaudal mamographic views. <span><span class="ref-journal">Med Phys. </span>2006;<span class="ref-vol">33</span>:3203–12.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17022213" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT49">49. <span class="element-citation">Jain AK, Duin RP, Mao J. Statistical pattern recognition: A review. <span><span class="ref-journal">IEEE Trans PAMI. </span>2000;<span class="ref-vol">22</span>:4–37.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT50">50. <span class="element-citation">Bezdek JC, Hall LO, Clarke LP. Review of MR image segmentation techniques using patternrecognition. <span><span class="ref-journal">Med Phys. </span>1993;<span class="ref-vol">20</span>:1033–48.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/8413011" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT51">51. <span class="element-citation">Frigui H, Krishnapuram R. A robust competitive clustering algorithm with application in computer vision. <span><span class="ref-journal">IEEE Trans PAMI. </span>1999;<span class="ref-vol">21</span>:450–65.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT52">52. <span class="element-citation">Tseng LY, Yang SB. A genetic approach to the automatic clustering problem. <span><span class="ref-journal">Patten Recog. </span>2001;<span class="ref-vol">34</span>:415–24.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT53">53. <span class="element-citation">Bandyopadhyay S. Simulated annealing using a reversible jump markov chain monte carlo algorithm for fuzzy clustering. <span><span class="ref-journal">IEEE Trans Knowledge Data Engg. </span>2005;<span class="ref-vol">17</span>:479–90.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT54">54. <span class="element-citation">Aurdal L, Omholt-Jensen T, Mala T, Edwin B. Segmentation of liver vessels as seen in MR and CT images. <span><span class="ref-journal">Int Congr Ser. </span>2004;<span class="ref-vol">1268</span>:201–6.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT55">55. <span class="element-citation">Bert A, Dmitriev I, Agliozzo S, Pietrosemoli N, Mandelkern M, Gallo T, et al.  An automatic method for colon segmentation in CT colonography. <span><span class="ref-journal">Comput Med Imaging Graphics. </span>2009;<span class="ref-vol">33</span>:325–31.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/19304454" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT56">56. <span class="element-citation">Poon CS, Braun M. Image segmentation by a deformable contour model incorporating region analysis. <span><span class="ref-journal">Physics Med Biol. </span>1997;<span class="ref-vol">42</span>:1833–41.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9308087" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT57">57. <span class="element-citation">Klinder T, Ostermann J, Ehm M, Franz A, Kneser R, Lorenz C. Automated model-based vertebra detection, identification, and segmentation in CT images. <span><span class="ref-journal">Med Image Anal. </span>2009;<span class="ref-vol">13</span>:471–82.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/19285910" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT58">58. <span class="element-citation">Sharma N, Ray AK, Sharma S, Shukla KK, Pradhan S, Aggarwal LM. Segmentation of medical images using simulated annealing based fuzzy C Means algorithm. <span><span class="ref-journal">Int J Biomed Engg Technol. </span>2009;<span class="ref-vol">2</span>:260–78.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT59">59. <span class="element-citation">Tabakov M. A fuzzy segmentation method for computed tomography images. <span><span class="ref-journal">Int J Intellig Inform Database Syst Technol Appl. </span>2007;<span class="ref-vol">1</span>:234–46.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT60">60. <span class="element-citation">Chen EL, Chung PC, Chen CL, Tsai HM, Chang CI. An automatic diagnostic system for CT liver image classification. <span><span class="ref-journal">IEEE Trans Biomed Engg. </span>1998;<span class="ref-vol">45</span>:783–93.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9609943" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT61">61. <span class="element-citation">Song T, Gasparovicc C, Andreasen N. A hybrid tissue segmentation approach for brain MR images. <span><span class="ref-journal">Med Biol Eng Comput. </span>2006;<span class="ref-vol">44</span>:242–9.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16937165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT62">62. <span class="element-citation">Liao L, Lin T, Li B. MRI brain image segmentation and bias field correction based on fast spatially constrained kernel clustering approach. <span><span class="ref-journal">Pattern Recog Lett. </span>2008;<span class="ref-vol">29</span>:1580–8.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT63">63. <span class="element-citation">Kuo WF, Lin CY, Yung-Nien Sun YN. Brain MR images segmentation using statistical ratio: Mapping between watershed and competitive Hopfield clustering network algorithms. <span><span class="ref-journal">Comput Met Prog Biomed. </span>2008;<span class="ref-vol">9</span>:191–8.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18555554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT64">64. <span class="element-citation">Cuadra MB, Craene MD, Duay V. Dense deformation field estimation for Atlas-based segmentation of pathological MR brain images. <span><span class="ref-journal">Comput Met Prog Biomed. </span>2006;<span class="ref-vol">84</span>:67–75.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16979256" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT65">65. <span class="element-citation">Sled JG, Pike GB. Standing-wave and RF penetration artifacts caused by elliptic geometry: An electrodynamic analysis of MRI. <span><span class="ref-journal">IEEE T Med Imag. </span>1998;<span class="ref-vol">17</span>:653–62.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9845320" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT66">66. <span class="element-citation">Garcia-Sebastian M, Hernandez C, Anjou A. Robustness of an adaptive MRI segmentation algorithm parametric intensityinhomogeneity modeling. <span><span class="ref-journal">Neurocomputing. </span>2009;<span class="ref-vol">72</span>:2146–52.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT67">67. <span class="element-citation">Chen Y, Zhang J, Macione J. An improved level set method for brain MR image segmentation and bias correction. <span><span class="ref-journal">Comput Med Imaging Graphics. </span>2009;<span class="ref-vol">33</span>:510–19.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/19481420" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT68">68. <span class="element-citation">Cuadra MB, Cammoun L, Butz T. Comparison and validation of tissue modelization and statistical classification methods in T1-weighted MR brain images. <span><span class="ref-journal">IEEE Trans Med Imaging. </span>2005;<span class="ref-vol">24</span>:1548–65.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16350916" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT69">69. <span class="element-citation">Zaidi H, Ruest T, Schoenahl F, Montandon ML. Comparative assessment of statistical brain MR image segmentation algorithms and their impact on partial volume correction in PET. <span><span class="ref-journal">Neuroimage. </span>2006;<span class="ref-vol">32</span>:1591–1607.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16828315" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT70">70. <span class="element-citation">Zhu C, Jiang T. Multicontext fuzzy clustering for separation of brain tissues in magnetic resonance images. <span><span class="ref-journal">Neuroimage. </span>2003;<span class="ref-vol">18</span>:685–96.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/12667846" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT71">71. <span class="element-citation">Ahmed MN, Yamany SM. A modified fuzzy C-means algorithm for bias field estimation and segmentation of MRI data. <span><span class="ref-journal">IEEE Trans Med Imaging. </span>2002;<span class="ref-vol">21</span>:193–9.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/11989844" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT72">72. <span class="element-citation">Pham DL, Prince JL. An adaptive fuzzy c-means algorithm for image segmentation in the presence of intensity in homogeneities. <span><span class="ref-journal">Pattern Rec Letters. </span>1999;<span class="ref-vol">14</span>:57–68.</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT73">73. <span class="element-citation">Shen S, Sandham W, Ranat M, Sterr A. MRI fuzzy segmentation of brain tissue using neighborhood attraction with neural-network optimization. <span><span class="ref-journal">IEEE Trans Inform Technol Biomed. </span>2005;<span class="ref-vol">9</span>:459–67.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16167700" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT74">74. <span class="element-citation">Vovk U, Pernuŝ, Franjo, Likar, Boŝtjan A review of methods for correction of intensity inhomogeneity in MRI. <span><span class="ref-journal">IEEE Trans Med Imaging. </span>2007;<span class="ref-vol">26</span>:405–21.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17354645" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#idm140507415367776title">Abstract</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#sec1-1title">Introduction</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#sec1-2title">Medical Imaging Modalities</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#sec1-3title">Segmentation</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#sec1-4title">Conclusion</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#idm140507479026928title">Footnotes</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#idm140507478829296title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Journal of Medical Physics</span> are provided here courtesy of <strong>Wolters Kluwer -- Medknow Publications</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/epub/">ePub (beta)</a></li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/?report=printable">Printer Friendly</a></li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" data-citationid="PMC2825001" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2825001%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Automated medical image segmentation techniques_files/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2825001%2F&amp;text=Automated%20medical%20image%20segmentation%20techniques" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Automated medical image segmentation techniques_files/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2825001%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Automated medical image segmentation techniques_files/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20177565" data-aiid="2825001" data-aid="2825001" data-iid="185087" data-domainid="976" data-domain="jmedphys" data-accid="PMC2825001" data-md5="7a7c5689b8a00e303450dac66a9b8743">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-4" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-4" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input type="hidden" name="absid" id="absid" value="2825001"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:&#39;#favUL&#39;,starExists:true,createCollectionUrl:&#39;#&#39;,manageCollectionsUrl:&#39;/myncbi/collections/&#39;"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="https://www.ncbi.nlm.nih.gov/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input type="submit" class="button_apply" name="addtocollection" value="1"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-5" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/26574298" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Bayesian segmentation of human facial tissue using 3D MR-CT information fusion, resolution enhancement and partial volume modelling.</a><span class="source">[Comput Methods Programs Biomed...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-6" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Şener E, Mumcuoglu EU, Hamcan S. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Comput Methods Programs Biomed. 2016 Feb; 124:31-44. Epub 2015 Oct 23.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/20129843" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Automatic segmentation of rotational x-ray images for anatomic intra-procedural surface generation in atrial fibrillation ablation procedures.</a><span class="source">[IEEE Trans Med Imaging. 2010]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Manzke R, Meyer C, Ecabert O, Peters J, Noordhoek NJ, Thiagalingam A, Reddy VY, Chan RC, Weese J. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">IEEE Trans Med Imaging. 2010 Feb; 29(2):260-72. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/20229871" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Development and validation of segmentation and interpolation techniques in sinograms for metal artifact suppression in CT.</a><span class="source">[Med Phys. 2010]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Veldkamp WJ, Joemai RM, van der Molen AJ, Geleijns J. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Med Phys. 2010 Feb; 37(2):620-8. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/11233837" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Aspects of MR image distortions in radiotherapy treatment planning.</a><span class="source">[Strahlenther Onkol. 2001]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Fransson A, Andreo P, Pötter R. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Strahlenther Onkol. 2001 Feb; 177(2):59-73. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/26025124" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A review of segmentation and deformable registration methods applied to adaptive cervical cancer radiation therapy treatment planning.</a><span class="source">[Artif Intell Med. 2015]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Ghose S, Holloway L, Lim K, Chan P, Veera J, Vinod SK, Liney G, Greer PB, Dowling J. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Artif Intell Med. 2015 Jun; 64(2):75-87. Epub 2015 May 16.</em>
</div>
</li>
</ul>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=20177565&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=20177565&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-11" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-11" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6011549/" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Radiation Exposure to the Personnel Performing Robotic Arm-Assisted Positron Emission Tomography/Computed Tomography-Guided Biopsies</a><span class="source">[Indian Journal of Nuclear Medi...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-12" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Lakhanpal T, Mittal BR, Kumar R, Watts A, Rana N, Singh H. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Indian Journal of Nuclear Medicine : IJNM : The Official Journal of the Society of Nuclear Medicine, India. 2018 Jul-Sep; 33(3): 209-213</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5980900/" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Lung Lesion Detection in CT Scan Images Using the Fuzzy Local Information Cluster Means (FLICM) Automatic Segmentation Algorithm and Back Propagation Network Classification</a><span class="source">[Asian Pacific Journal of Cance...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Lavanya M, Kannan PM. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Asian Pacific Journal of Cancer Prevention : APJCP. 2017; 18(12): 3395-3399</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5938243/" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Quantitative image analysis for evaluation of tumor response in clinical oncology</a><span class="source">[Chronic Diseases and Translati...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Cai WL, Hong GB. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Chronic Diseases and Translational Medicine. 2018 Mar 8; 4(1): 18-28</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5875567/" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A comparative study of automatic image segmentation algorithms for target tracking in MR‐IGRT</a><span class="source">[Journal of Applied Clinical Me...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Feng Y, Kawrakow I, Olsen J, Parikh PJ, Noel C, Wooten O, Du D, Mutic S, Hu Y. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Journal of Applied Clinical Medical Physics. 2016 Mar 8; 17(2): 441-460</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5816252/" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A Re-Engineered Software Interface and Workflow for the Open-Source SimVascular Cardiovascular Modeling Package</a><span class="source">[Journal of Biomechanical Engin...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Lan H, Updegrove A, Wilson NM, Maher GD, Shadden SC, Marsden AL. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Journal of Biomechanical Engineering. 2018 Jan 25; 140(2): 0245011-02450111</em>
</div>
</li>
</ul>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/citedby/" ref="reftype=relart&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-17" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-17" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/?Db=pccompound&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_pccompound&amp;IdsFromResult=2825001" ref="reftype=Compound&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|PubChem%20Compound&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Compound</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-18" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Compound<div class="brieflinkpopdesc">PubChem Compound links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/20177565/" ref="reftype=PubMed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/?Db=pcsubstance&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_pcsubstance&amp;IdsFromResult=2825001" ref="reftype=Substance&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|PubChem%20Substance&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Substance</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Substance<div class="brieflinkpopdesc">PubChem Substance links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=2825001" ref="reftype=Taxonomy&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-21" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-22" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-22" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState(&#39;ClearHT&#39;)">Clear</a><a href="javascript:historyDisplayState(&#39;HTOff&#39;)" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState(&#39;HTOn&#39;)" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=5b88149939a66e2262188f2f">Automated medical image segmentation techniques</a><div class="ralinkpop offscreen_noflow">Automated medical image segmentation techniques<div class="brieflinkpopdesc">Journal of Medical Physics. Jan-Mar 2010; 35(1)3</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=5b85a6ea75f66dd394b3386f">Recurrent Urinary Tract Infections Management in Women</a><div class="ralinkpop offscreen_noflow">Recurrent Urinary Tract Infections Management in Women<div class="brieflinkpopdesc">Sultan Qaboos University Medical Journal. 2013 Aug; 13(3)359</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=5b7f1eb539a66eb31038af77">Deep Convolutional Neural Networks for Computer-Aided Detection: CNN Architectur...</a><div class="ralinkpop offscreen_noflow">Deep Convolutional Neural Networks for Computer-Aided Detection: CNN Architectures, Dataset Characteristics and Transfer Learning<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2016 May; 35(5)1285</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=5b7efade39a66eb310caad3b">Activation of subventricular zone stem cells after neuronal injury.</a><div class="ralinkpop offscreen_noflow">Activation of subventricular zone stem cells after neuronal injury.<div class="brieflinkpopdesc">Cell Tissue Res. 2008 Jan;331(1):337-45. Epub 2007 Aug 11.</div>
</div>
<div class="tertiary">PubMed</div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=5b44109ebbb053f14bee162d">Medical Image File Formats</a><div class="ralinkpop offscreen_noflow">Medical Image File Formats<div class="brieflinkpopdesc">Journal of Digital Imaging. 2014 Apr; 27(2)200</div>
</div>
<div class="tertiary"></div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState(&#39;HTOn&#39;)">Turn recording back on</a></p>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
<div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p45" name="crb--__p45" rid="__p45" style="position: absolute; visibility: visible; width: 370px; top: 4676px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-23"><ul><li class="two_line" reference_id="CIT4" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18483138/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">CT artifact recognition for the nuclear technologist.</a><span class="one_line_source">[J Nucl Med Technol.  2008]</span><div class="alt-note">
<div class="authors">Popilock R, Sandrasagaren K, Harris L, Kaser KA</div>
<div class="citation">J Nucl Med Technol. 2008 Jun; 36(2):79-81.</div>
</div>
</li></ul><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/" class="seemore">See links ...</a></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p57" name="crb--__p57" rid="__p57" style="position: absolute; visibility: visible; width: 370px; top: 5646px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-24"><ul><li class="two_line" reference_id="CIT5">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18215825/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Object recognition in brain CT-scans: knowledge-based fusion of data from multiple feature extractors.</a><span class="one_line_source">[IEEE Trans Med Imaging.  1995]</span><div class="alt-note">
<div class="authors">Li H, Deklerck R, De Cuyper B, Hermanus A, Nyssen E, Cornelis J</div>
<div class="citation">IEEE Trans Med Imaging. 1995; 14(2):212-29.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p69" name="crb--__p69" rid="__p69" style="position: absolute; visibility: visible; width: 370px; top: 6235px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-25"><ul><li class="two_line" reference_id="CIT10">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/11701515/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Current methods in medical image segmentation.</a><span class="one_line_source">[Annu Rev Biomed Eng.  2000]</span><div class="alt-note">
<div class="authors">Pham DL, Xu C, Prince JL</div>
<div class="citation">Annu Rev Biomed Eng. 2000; 2():315-37.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p130" name="crb--__p130" rid="__p130" style="position: absolute; visibility: visible; width: 370px; top: 12879px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-26"><ul><li class="two_line" reference_id="CIT31">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/7207603/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Textons, the elements of texture perception, and their interactions.</a><span class="one_line_source">[Nature.  1981]</span><div class="alt-note">
<div class="authors">Julesz B</div>
<div class="citation">Nature. 1981 Mar 12; 290(5802):91-7.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p139" name="crb--__p139" rid="__p139" style="position: absolute; visibility: visible; width: 370px; top: 13533px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-27"><ul>
<li class="two_line" reference_id="CIT40" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18614335/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Medical image analysis of 3D CT images based on extension of Haralick texture features.</a><span class="one_line_source">[Comput Med Imaging Graph.  2008]</span><div class="alt-note">
<div class="authors">Tesar L, Shimizu A, Smutek D, Kobatake H, Nawano S</div>
<div class="citation">Comput Med Imaging Graph. 2008 Sep; 32(6):513-20.</div>
</div>
</li>
<li class="two_line" reference_id="CIT42" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/10504097/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Abdominal organ segmentation using texture transforms and a Hopfield neural network.</a><span class="one_line_source">[IEEE Trans Med Imaging.  1999]</span><div class="alt-note">
<div class="authors">Koss JE, Newman FD, Johnson TK, Kirch DL</div>
<div class="citation">IEEE Trans Med Imaging. 1999 Jul; 18(7):640-8.</div>
</div>
</li>
</ul><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/" class="seemore">See links ...</a></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p149" name="crb--__p149" rid="__p149" style="display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-28"><ul><li class="expanded" reference_id="CIT10">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/11701515/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Current methods in medical image segmentation.</a><span class="one_line_source">[Annu Rev Biomed Eng.  2000]</span><div class="alt-note">
<div class="authors">Pham DL, Xu C, Prince JL</div>
<div class="citation">Annu Rev Biomed Eng. 2000; 2():315-37.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p151" name="crb--__p151" rid="__p151" style="position: absolute; visibility: visible; width: 370px; top: 14330px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-29"><ul>
<li class="two_line" reference_id="CIT43">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/7791545/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> MRI segmentation: methods and applications.</a><span class="one_line_source">[Magn Reson Imaging.  1995]</span><div class="alt-note">
<div class="authors">Clarke LP, Velthuizen RP, Camacho MA, Heine JJ, Vaidyanathan M, Hall LO, Thatcher RW, Silbiger ML</div>
<div class="citation">Magn Reson Imaging. 1995; 13(3):343-68.</div>
</div>
</li>
<li class="two_line" reference_id="CIT44">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/11516707/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Interaction in the segmentation of medical images: a survey.</a><span class="one_line_source">[Med Image Anal.  2001]</span><div class="alt-note">
<div class="authors">Olabarriaga SD, Smeulders AW</div>
<div class="citation">Med Image Anal. 2001 Jun; 5(2):127-42.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p156" name="crb--__p156" rid="__p156" style="position: absolute; visibility: visible; width: 370px; top: 14685px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-30"><ul>
<li class="two_line" reference_id="CIT45">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17572068/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Segmentation and grading of brain tumors on apparent diffusion coefficient images using self-organizing maps.</a><span class="one_line_source">[Comput Med Imaging Graph.  2007]</span><div class="alt-note">
<div class="authors">Vijayakumar C, Damayanti G, Pant R, Sreedhar CM</div>
<div class="citation">Comput Med Imaging Graph. 2007 Oct; 31(7):473-84.</div>
</div>
</li>
<li class="two_line" reference_id="CIT47">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/14518728/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A computer-aided diagnostic system to characterize CT focal liver lesions: design and optimization of a neural network classifier.</a><span class="one_line_source">[IEEE Trans Inf Technol Biomed.  2003]</span><div class="alt-note">
<div class="authors">Gletsos M, Mougiakakou SG, Matsopoulos GK, Nikita KS, Nikita AS, Kelekis D</div>
<div class="citation">IEEE Trans Inf Technol Biomed. 2003 Sep; 7(3):153-62.</div>
</div>
</li>
<li class="two_line" reference_id="CIT48">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17022213/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Finding corresponding regions of interest in mediolateral oblique and craniocaudal mammographic views.</a><span class="one_line_source">[Med Phys.  2006]</span><div class="alt-note">
<div class="authors">van Engeland S, Timp S, Karssemeijer N</div>
<div class="citation">Med Phys. 2006 Sep; 33(9):3203-12.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p157" name="crb--__p157" rid="__p157" style="position: absolute; visibility: visible; width: 370px; top: 14988px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-31"><ul><li class="two_line" reference_id="CIT50">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/8413011/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Review of MR image segmentation techniques using pattern recognition.</a><span class="one_line_source">[Med Phys.  1993]</span><div class="alt-note">
<div class="authors">Bezdek JC, Hall LO, Clarke LP</div>
<div class="citation">Med Phys. 1993 Jul-Aug; 20(4):1033-48.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p158" name="crb--__p158" rid="__p158" style="position: absolute; visibility: visible; width: 370px; top: 15104px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-32"><ul><li class="two_line" reference_id="CIT50">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/8413011/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Review of MR image segmentation techniques using pattern recognition.</a><span class="one_line_source">[Med Phys.  1993]</span><div class="alt-note">
<div class="authors">Bezdek JC, Hall LO, Clarke LP</div>
<div class="citation">Med Phys. 1993 Jul-Aug; 20(4):1033-48.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p163" name="crb--__p163" rid="__p163" style="display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-33"><ul><li class="expanded" reference_id="CIT55">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/19304454/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An automatic method for colon segmentation in CT colonography.</a><span class="one_line_source">[Comput Med Imaging Graph.  2009]</span><div class="alt-note">
<div class="authors">Bert A, Dmitriev I, Agliozzo S, Pietrosemoli N, Mandelkern M, Gallo T, Regge D</div>
<div class="citation">Comput Med Imaging Graph. 2009 Jun; 33(4):325-31.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p164" name="crb--__p164" rid="__p164" style="display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-34"><ul>
<li class="expanded" reference_id="CIT56">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9308087/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Image segmentation by a deformable contour model incorporating region analysis.</a><span class="one_line_source">[Phys Med Biol.  1997]</span><div class="alt-note">
<div class="authors">Poon CS, Braun M</div>
<div class="citation">Phys Med Biol. 1997 Sep; 42(9):1833-41.</div>
</div>
</li>
<li class="expanded" reference_id="CIT57">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/19285910/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Automated model-based vertebra detection, identification, and segmentation in CT images.</a><span class="one_line_source">[Med Image Anal.  2009]</span><div class="alt-note">
<div class="authors">Klinder T, Ostermann J, Ehm M, Franz A, Kneser R, Lorenz C</div>
<div class="citation">Med Image Anal. 2009 Jun; 13(3):471-82.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p166" name="crb--__p166" rid="__p166" style="display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-35"><ul>
<li class="expanded" reference_id="CIT45">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17572068/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Segmentation and grading of brain tumors on apparent diffusion coefficient images using self-organizing maps.</a><span class="one_line_source">[Comput Med Imaging Graph.  2007]</span><div class="alt-note">
<div class="authors">Vijayakumar C, Damayanti G, Pant R, Sreedhar CM</div>
<div class="citation">Comput Med Imaging Graph. 2007 Oct; 31(7):473-84.</div>
</div>
</li>
<li class="expanded" reference_id="CIT47">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/14518728/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A computer-aided diagnostic system to characterize CT focal liver lesions: design and optimization of a neural network classifier.</a><span class="one_line_source">[IEEE Trans Inf Technol Biomed.  2003]</span><div class="alt-note">
<div class="authors">Gletsos M, Mougiakakou SG, Matsopoulos GK, Nikita KS, Nikita AS, Kelekis D</div>
<div class="citation">IEEE Trans Inf Technol Biomed. 2003 Sep; 7(3):153-62.</div>
</div>
</li>
<li class="expanded" reference_id="CIT60">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9609943/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An automatic diagnostic system for CT liver image classification.</a><span class="one_line_source">[IEEE Trans Biomed Eng.  1998]</span><div class="alt-note">
<div class="authors">Chen EL, Chung PC, Chen CL, Tsai HM, Chang CI</div>
<div class="citation">IEEE Trans Biomed Eng. 1998 Jun; 45(6):783-94.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p167" name="crb--__p167" rid="__p167" style="position: absolute; visibility: visible; width: 370px; top: 15747px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-36"><ul>
<li class="two_line" reference_id="CIT61">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16937165/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A hybrid tissue segmentation approach for brain MR images.</a><span class="one_line_source">[Med Biol Eng Comput.  2006]</span><div class="alt-note">
<div class="authors">Song T, Gasparovic C, Andreasen N, Bockholt J, Jamshidi M, Lee RR, Huang M</div>
<div class="citation">Med Biol Eng Comput. 2006 Mar; 44(3):242-9.</div>
</div>
</li>
<li class="two_line" reference_id="CIT64">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16979256/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Dense deformation field estimation for atlas-based segmentation of pathological MR brain images.</a><span class="one_line_source">[Comput Methods Programs Biomed.  2006]</span><div class="alt-note">
<div class="authors">Bach Cuadra M, De Craene M, Duay V, Macq B, Pollo C, Thiran JP</div>
<div class="citation">Comput Methods Programs Biomed. 2006 Dec; 84(2-3):66-75.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p169" name="crb--__p169" rid="__p169" style="position: absolute; visibility: visible; width: 370px; top: 15998px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-37"><ul>
<li class="two_line" reference_id="CIT65">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9845320/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Standing-wave and RF penetration artifacts caused by elliptic geometry: an electrodynamic analysis of MRI.</a><span class="one_line_source">[IEEE Trans Med Imaging.  1998]</span><div class="alt-note">
<div class="authors">Sled JG, Pike GB</div>
<div class="citation">IEEE Trans Med Imaging. 1998 Aug; 17(4):653-62.</div>
</div>
</li>
<li class="two_line" reference_id="CIT67">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/19481420/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">An improved level set method for brain MR images segmentation and bias correction.</a><span class="one_line_source">[Comput Med Imaging Graph.  2009]</span><div class="alt-note">
<div class="authors">Chen Y, Zhang J, Macione J</div>
<div class="citation">Comput Med Imaging Graph. 2009 Oct; 33(7):510-9.</div>
</div>
</li>
<li class="two_line" reference_id="CIT68">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16350916/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Comparison and validation of tissue modelization and statistical classification methods in T1-weighted MR brain images.</a><span class="one_line_source">[IEEE Trans Med Imaging.  2005]</span><div class="alt-note">
<div class="authors">Cuadra MB, Cammoun L, Butz T, Cuisenaire O, Thiran JP</div>
<div class="citation">IEEE Trans Med Imaging. 2005 Dec; 24(12):1548-65.</div>
</div>
</li>
<li class="two_line" reference_id="CIT69" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16828315/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Comparative assessment of statistical brain MR image segmentation algorithms and their impact on partial volume correction in PET.</a><span class="one_line_source">[Neuroimage.  2006]</span><div class="alt-note">
<div class="authors">Zaidi H, Ruest T, Schoenahl F, Montandon ML</div>
<div class="citation">Neuroimage. 2006 Oct 1; 32(4):1591-607.</div>
</div>
</li>
<li class="two_line" reference_id="CIT70" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/12667846/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Multicontext fuzzy clustering for separation of brain tissues in magnetic resonance images.</a><span class="one_line_source">[Neuroimage.  2003]</span><div class="alt-note">
<div class="authors">Zhu C, Jiang T</div>
<div class="citation">Neuroimage. 2003 Mar; 18(3):685-96.</div>
</div>
</li>
<li class="two_line" reference_id="CIT73" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16167700/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">MRI fuzzy segmentation of brain tissue using neighborhood attraction with neural-network optimization.</a><span class="one_line_source">[IEEE Trans Inf Technol Biomed.  2005]</span><div class="alt-note">
<div class="authors">Shen S, Sandham W, Granat M, Sterr A</div>
<div class="citation">IEEE Trans Inf Technol Biomed. 2005 Sep; 9(3):459-67.</div>
</div>
</li>
<li class="two_line" reference_id="CIT74" style="display: none;">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17354645/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> A review of methods for correction of intensity inhomogeneity in MRI.</a><span class="one_line_source">[IEEE Trans Med Imaging.  2007]</span><div class="alt-note">
<div class="authors">Vovk U, Pernus F, Likar B</div>
<div class="citation">IEEE Trans Med Imaging. 2007 Mar; 26(3):405-21.</div>
</div>
</li>
</ul><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/" class="seemore">See more ...</a></div>
</div>
</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="./Automated medical image segmentation techniques_files/jquery.scrollTo-1.4.2.js.download"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"><div id="body-link-popper-CIT1" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="width: 30em; top: -100px; left: -100px; display: none;" aria-live="assertive" aria-hidden="true">1. <span class="element-citation">Withey DJ, Koles ZJ. Three generations of medical image segmentation: Methods and available software. <span><span class="ref-journal">Int J Bioelectromag. </span>2007;<span class="ref-vol">9</span>:67–8.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT1">Ref list</a>]</div><div id="body-link-popper-CIT2" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">2. <span class="element-citation">Prince JL, Links JM. Medical imaging signals and system. <span class="ref-journal">Pearson Education.</span> 2006. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT2">Ref list</a>]</div><div id="body-link-popper-CIT3" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">3. <span class="element-citation">Macovski A.  <span class="ref-journal">Medical imaging systems.</span> Prentice-Hall; 1983. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT3">Ref list</a>]</div><div id="body-link-popper-CIT4" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">CT artifact recognition for the nuclear technologist.<div class="authors">Popilock R, Sandrasagaren K, Harris L, Kaser KA</div><div class="citation">J Nucl Med Technol. 2008 Jun; 36(2):79-81.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18483138/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT4">Ref list</a>]</p></div><div id="body-link-popper-CIT5" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Object recognition in brain CT-scans: knowledge-based fusion of data from multiple feature extractors.<div class="authors">Li H, Deklerck R, De Cuyper B, Hermanus A, Nyssen E, Cornelis J</div><div class="citation">IEEE Trans Med Imaging. 1995; 14(2):212-29.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18215825/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT5">Ref list</a>]</p></div><div id="body-link-popper-CIT6" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">6. <span class="element-citation">Pham DL, Prince JL, Dagher AP, Xu C. An automated technique for statistical characterization of brain tissues in magnetic resonance imaging. <span><span class="ref-journal">Int J Patt Rec Art Intel. </span>1997;<span class="ref-vol">11</span>:1189–211.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT6">Ref list</a>]</div><div id="body-link-popper-CIT7" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">7. <span class="element-citation">Gonzalez RC, Woods RE.  <span class="ref-journal">Digital image processing.</span> 2nd ed. 2004.  Pearson Education.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT7">Ref list</a>]</div><div id="body-link-popper-CIT9" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">9. <span class="element-citation">Pal NR, Pal SH. A review on image segmentation techniques. <span><span class="ref-journal">Pattern Recog. </span>1993;<span class="ref-vol">26</span>:1277–94.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT9">Ref list</a>]</div><div id="body-link-popper-CIT10" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="width: 30em; top: -100px; left: -100px; display: none;" aria-live="assertive" aria-hidden="true"><span class="invert">Review</span> Current methods in medical image segmentation.<div class="authors">Pham DL, Xu C, Prince JL</div><div class="citation">Annu Rev Biomed Eng. 2000; 2():315-37.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/11701515/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT10">Ref list</a>]</p></div><div id="body-link-popper-CIT11" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">11. <span class="element-citation">Ramesh N, Yoo JH, Sethi IK. Thresholding based on histogram approximation. <span><span class="ref-journal">IEEE Proc Vision Image Signal Proc. </span>1995;<span class="ref-vol">142</span>:271–9.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT11">Ref list</a>]</div><div id="body-link-popper-CIT12" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">12. <span class="element-citation">Sharma N, Ray AK.  <span class="ref-journal">Proc. of Int. Conf. on Mathematical Biology'.</span> Mathematical Biology recent trends by ANAMAYA Publishers; 2006. Computer aided segmentation of medical images based on hybridized approach of edge and region based techniques; pp. 150–5.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT12">Ref list</a>]</div><div id="body-link-popper-CIT13" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">13. <span class="element-citation">Wang Z, Gierriero A, Sario M. Comparison of several approaches for segmentation of texture images. <span><span class="ref-journal">Pattern Recog Lett. </span>1996;<span class="ref-vol">17</span>:509–21.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT13">Ref list</a>]</div><div id="body-link-popper-CIT14" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">14. <span class="element-citation">Sharma N, Ray AK, Sharma S, Shukla KK, Pradhan S, Aggarwal LM. Segmentation and classification of medical images using texture-primitive features: Application of BAM-type artificial neural network. <span><span class="ref-journal">J Med Physics. </span>2008;<span class="ref-vol">33</span>:119–26.</span> <span class="nowrap ref pmc">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2772042/">PMC free article</a>]</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/19893702" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT14">Ref list</a>]</div><div id="body-link-popper-CIT15" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">15. <span class="element-citation">Lai CC. A novel image segmentation approach based on particle swarm optimization. <span><span class="ref-journal">IEICE Trans Fundamentals. </span>2006;<span class="ref-vol">E89A</span>:324–7.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT15">Ref list</a>]</div><div id="body-link-popper-CIT17" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">17. <span class="element-citation">Frank RJ, Grabowski TJ, Damasio H.  <span class="ref-journal">In Abstracts, 25th Annual meeting, Society for Neuroscience, Society for Neuroscience.</span> Washington, DC: 1995. Voxelvise percentage tissue segmentation of human brain Magnetic resonance images; p. 694.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT17">Ref list</a>]</div><div id="body-link-popper-CIT18" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">18. <span class="element-citation">Hancock ER, Kittler J. Edge labeling using dictionary-based relaxation. <span><span class="ref-journal">IEEE Trans PAMI. </span>1990;<span class="ref-vol">12</span>:165–81.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT18">Ref list</a>]</div><div id="body-link-popper-CIT19" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">19. <span class="element-citation">Liow YT. A contour tracing algorithm that preserves common boundaries between regions. <span><span class="ref-journal">CVGIP Image Understanding. </span>1991;<span class="ref-vol">53</span>:313–21.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT19">Ref list</a>]</div><div id="body-link-popper-CIT21" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">21. <span class="element-citation">Ney H. A comparative study of two search strategies for connected word recognition Dynamic programming and heuristic search. <span><span class="ref-journal">IEEE Trans PAMI. </span>1992;<span class="ref-vol">14</span>:586–95.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT21">Ref list</a>]</div><div id="body-link-popper-CIT22" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">22. <span class="element-citation">Xu L, Oja E. Randomized Hough transform (RHT): Basic mechanism and algorithms, and computational complexities. <span><span class="ref-journal">CVGIP Image Comput. </span>1993;<span class="ref-vol">57</span>:131–54.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT22">Ref list</a>]</div><div id="body-link-popper-CIT23" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">23. <span class="element-citation">Kalvian H, Hirvonen P, Xu L, Oja E. Probabilistic and non-probabilistic Hough transform: Overview and comparisons. <span><span class="ref-journal">Image Comput. </span>1995;<span class="ref-vol">13</span>:239–52.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT23">Ref list</a>]</div><div id="body-link-popper-CIT24" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">24. <span class="element-citation">Sonka M, Hlavac V, Boyle R.  <span class="ref-journal">Image processing, analysis and machine vision.</span> Singapore: Thomson Learning; 1999. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT24">Ref list</a>]</div><div id="body-link-popper-CIT25" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">25. <span class="element-citation">Ng HP, Ong SH, Foong KW, Nowinski WL. An improved watershed algorithm for medical image segmentation. <span class="ref-journal">Proceedings 12th International Conference on Biomedical Engineering.</span> 2005. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT25">Ref list</a>]</div><div id="body-link-popper-CIT28" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">28. <span class="element-citation">Lopez AM. Evaluation of methods for ridge and valley detection. <span><span class="ref-journal">IEEE Trans PAMI. </span>1999;<span class="ref-vol">21</span>:327.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT28">Ref list</a>]</div><div id="body-link-popper-CIT29" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">29. <span class="element-citation">Gevers T, Smeulders AW.  <span class="ref-journal">Comp Vision Pattern Recognition IEEE Computer Society.</span> Los Almitos, CA: 1997. Combining region splitting and edge detection through guided Delaunay image subdivision; pp. 1021–6.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT29">Ref list</a>]</div><div id="body-link-popper-CIT30" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">30. <span class="element-citation">Haralick RM. Statistical and structural approaches to texture. <span><span class="ref-journal">Proc IEEE. </span>1979;<span class="ref-vol">67</span>:786–804.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT30">Ref list</a>]</div><div id="body-link-popper-CIT31" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Textons, the elements of texture perception, and their interactions.<div class="authors">Julesz B</div><div class="citation">Nature. 1981 Mar 12; 290(5802):91-7.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7207603/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT31">Ref list</a>]</p></div><div id="body-link-popper-CIT32" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">32. <span class="element-citation">Fukunaga K.  <span class="ref-journal">Introduction to statistical pattern recognition.</span> 2nd ed. Academic Press; 1990. </span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT32">Ref list</a>]</div><div id="body-link-popper-CIT33" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">33. <span class="element-citation">Pavilidis T.  Structural description and graph grammar. In: Chang SK, Fu KS, editors. <span class="ref-journal">Pictorial information systems.</span> Springer Verlag Berlin: 1980. pp. 86–103.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT33">Ref list</a>]</div><div id="body-link-popper-CIT34" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">34. <span class="element-citation">Argenti F, Alparone L, Benelli G. Fast algorithm for texture analysis using co-occurrence matrices. <span><span class="ref-journal">IEE Proc Part F: Radar Signal Proc. </span>1990;<span class="ref-vol">137</span>:443–8.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT34">Ref list</a>]</div><div id="body-link-popper-CIT35" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">35. <span class="element-citation">Haralick RM, Shanmugan K, Dinstein I. Textural features for image classification. <span><span class="ref-journal">IEEE Trans Syst Man Cybern. </span>1973;<span class="ref-vol">3</span>:610–22.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT35">Ref list</a>]</div><div id="body-link-popper-CIT36" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">36. <span class="element-citation">Chu CC, Aggarwal JK. The integration of image segmentation maps using region and edge information. <span><span class="ref-journal">IEEE Trans Pattern Anal Mach Intellig. </span>1993;<span class="ref-vol">15</span>:1241–52.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT36">Ref list</a>]</div><div id="body-link-popper-CIT37" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">37. <span class="element-citation">Chaudhuri BB, Sarkar N. Texture segmentation using fractal dimension. <span><span class="ref-journal">IEEE Trans PAMI. </span>1995;<span class="ref-vol">17</span>:72–7.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT37">Ref list</a>]</div><div id="body-link-popper-CIT38" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">38. <span class="element-citation">Lu SY, Fu KS. A syntactic approach to texture analysis. <span><span class="ref-journal">Comput Graphics Image Proc. </span>1978;<span class="ref-vol">7</span>:303–30.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT38">Ref list</a>]</div><div id="body-link-popper-CIT39" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">39. <span class="element-citation">Duda RO, Hart PE, Stork DG.  <span class="ref-journal">Pattern classification.</span> Singapore: Wiley; 2001. pp. 350–93.</span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT39">Ref list</a>]</div><div id="body-link-popper-CIT40" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Medical image analysis of 3D CT images based on extension of Haralick texture features.<div class="authors">Tesar L, Shimizu A, Smutek D, Kobatake H, Nawano S</div><div class="citation">Comput Med Imaging Graph. 2008 Sep; 32(6):513-20.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/18614335/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT40">Ref list</a>]</p></div><div id="body-link-popper-CIT42" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Abdominal organ segmentation using texture transforms and a Hopfield neural network.<div class="authors">Koss JE, Newman FD, Johnson TK, Kirch DL</div><div class="citation">IEEE Trans Med Imaging. 1999 Jul; 18(7):640-8.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/10504097/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT42">Ref list</a>]</p></div><div id="body-link-popper-CIT43" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="invert">Review</span> MRI segmentation: methods and applications.<div class="authors">Clarke LP, Velthuizen RP, Camacho MA, Heine JJ, Vaidyanathan M, Hall LO, Thatcher RW, Silbiger ML</div><div class="citation">Magn Reson Imaging. 1995; 13(3):343-68.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/7791545/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT43">Ref list</a>]</p></div><div id="body-link-popper-CIT44" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Interaction in the segmentation of medical images: a survey.<div class="authors">Olabarriaga SD, Smeulders AW</div><div class="citation">Med Image Anal. 2001 Jun; 5(2):127-42.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/11516707/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT44">Ref list</a>]</p></div><div id="body-link-popper-CIT45" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Segmentation and grading of brain tumors on apparent diffusion coefficient images using self-organizing maps.<div class="authors">Vijayakumar C, Damayanti G, Pant R, Sreedhar CM</div><div class="citation">Comput Med Imaging Graph. 2007 Oct; 31(7):473-84.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17572068/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT45">Ref list</a>]</p></div><div id="body-link-popper-CIT47" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A computer-aided diagnostic system to characterize CT focal liver lesions: design and optimization of a neural network classifier.<div class="authors">Gletsos M, Mougiakakou SG, Matsopoulos GK, Nikita KS, Nikita AS, Kelekis D</div><div class="citation">IEEE Trans Inf Technol Biomed. 2003 Sep; 7(3):153-62.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/14518728/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT47">Ref list</a>]</p></div><div id="body-link-popper-CIT48" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Finding corresponding regions of interest in mediolateral oblique and craniocaudal mammographic views.<div class="authors">van Engeland S, Timp S, Karssemeijer N</div><div class="citation">Med Phys. 2006 Sep; 33(9):3203-12.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17022213/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT48">Ref list</a>]</p></div><div id="body-link-popper-CIT49" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">49. <span class="element-citation">Jain AK, Duin RP, Mao J. Statistical pattern recognition: A review. <span><span class="ref-journal">IEEE Trans PAMI. </span>2000;<span class="ref-vol">22</span>:4–37.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT49">Ref list</a>]</div><div id="body-link-popper-CIT50" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="invert">Review</span> Review of MR image segmentation techniques using pattern recognition.<div class="authors">Bezdek JC, Hall LO, Clarke LP</div><div class="citation">Med Phys. 1993 Jul-Aug; 20(4):1033-48.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/8413011/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT50">Ref list</a>]</p></div><div id="body-link-popper-CIT51" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">51. <span class="element-citation">Frigui H, Krishnapuram R. A robust competitive clustering algorithm with application in computer vision. <span><span class="ref-journal">IEEE Trans PAMI. </span>1999;<span class="ref-vol">21</span>:450–65.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT51">Ref list</a>]</div><div id="body-link-popper-CIT53" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">53. <span class="element-citation">Bandyopadhyay S. Simulated annealing using a reversible jump markov chain monte carlo algorithm for fuzzy clustering. <span><span class="ref-journal">IEEE Trans Knowledge Data Engg. </span>2005;<span class="ref-vol">17</span>:479–90.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT53">Ref list</a>]</div><div id="body-link-popper-CIT54" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">54. <span class="element-citation">Aurdal L, Omholt-Jensen T, Mala T, Edwin B. Segmentation of liver vessels as seen in MR and CT images. <span><span class="ref-journal">Int Congr Ser. </span>2004;<span class="ref-vol">1268</span>:201–6.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT54">Ref list</a>]</div><div id="body-link-popper-CIT55" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">55. <span class="element-citation">Bert A, Dmitriev I, Agliozzo S, Pietrosemoli N, Mandelkern M, Gallo T, et al.  An automatic method for colon segmentation in CT colonography. <span><span class="ref-journal">Comput Med Imaging Graphics. </span>2009;<span class="ref-vol">33</span>:325–31.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/19304454" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT55">Ref list</a>]</div><div id="body-link-popper-CIT56" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">56. <span class="element-citation">Poon CS, Braun M. Image segmentation by a deformable contour model incorporating region analysis. <span><span class="ref-journal">Physics Med Biol. </span>1997;<span class="ref-vol">42</span>:1833–41.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9308087" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT56">Ref list</a>]</div><div id="body-link-popper-CIT57" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">57. <span class="element-citation">Klinder T, Ostermann J, Ehm M, Franz A, Kneser R, Lorenz C. Automated model-based vertebra detection, identification, and segmentation in CT images. <span><span class="ref-journal">Med Image Anal. </span>2009;<span class="ref-vol">13</span>:471–82.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/19285910" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT57">Ref list</a>]</div><div id="body-link-popper-CIT58" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">58. <span class="element-citation">Sharma N, Ray AK, Sharma S, Shukla KK, Pradhan S, Aggarwal LM. Segmentation of medical images using simulated annealing based fuzzy C Means algorithm. <span><span class="ref-journal">Int J Biomed Engg Technol. </span>2009;<span class="ref-vol">2</span>:260–78.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT58">Ref list</a>]</div><div id="body-link-popper-CIT59" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">59. <span class="element-citation">Tabakov M. A fuzzy segmentation method for computed tomography images. <span><span class="ref-journal">Int J Intellig Inform Database Syst Technol Appl. </span>2007;<span class="ref-vol">1</span>:234–46.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT59">Ref list</a>]</div><div id="body-link-popper-CIT60" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">60. <span class="element-citation">Chen EL, Chung PC, Chen CL, Tsai HM, Chang CI. An automatic diagnostic system for CT liver image classification. <span><span class="ref-journal">IEEE Trans Biomed Engg. </span>1998;<span class="ref-vol">45</span>:783–93.</span>  <span class="nowrap ref pubmed">[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9609943" target="pmc_ext" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal">PubMed</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT60">Ref list</a>]</div><div id="body-link-popper-CIT61" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A hybrid tissue segmentation approach for brain MR images.<div class="authors">Song T, Gasparovic C, Andreasen N, Bockholt J, Jamshidi M, Lee RR, Huang M</div><div class="citation">Med Biol Eng Comput. 2006 Mar; 44(3):242-9.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16937165/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT61">Ref list</a>]</p></div><div id="body-link-popper-CIT64" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Dense deformation field estimation for atlas-based segmentation of pathological MR brain images.<div class="authors">Bach Cuadra M, De Craene M, Duay V, Macq B, Pollo C, Thiran JP</div><div class="citation">Comput Methods Programs Biomed. 2006 Dec; 84(2-3):66-75.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16979256/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT64">Ref list</a>]</p></div><div id="body-link-popper-CIT65" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Standing-wave and RF penetration artifacts caused by elliptic geometry: an electrodynamic analysis of MRI.<div class="authors">Sled JG, Pike GB</div><div class="citation">IEEE Trans Med Imaging. 1998 Aug; 17(4):653-62.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9845320/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT65">Ref list</a>]</p></div><div id="body-link-popper-CIT66" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">66. <span class="element-citation">Garcia-Sebastian M, Hernandez C, Anjou A. Robustness of an adaptive MRI segmentation algorithm parametric intensityinhomogeneity modeling. <span><span class="ref-journal">Neurocomputing. </span>2009;<span class="ref-vol">72</span>:2146–52.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT66">Ref list</a>]</div><div id="body-link-popper-CIT67" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">An improved level set method for brain MR images segmentation and bias correction.<div class="authors">Chen Y, Zhang J, Macione J</div><div class="citation">Comput Med Imaging Graph. 2009 Oct; 33(7):510-9.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/19481420/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT67">Ref list</a>]</p></div><div id="body-link-popper-CIT68" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Comparison and validation of tissue modelization and statistical classification methods in T1-weighted MR brain images.<div class="authors">Cuadra MB, Cammoun L, Butz T, Cuisenaire O, Thiran JP</div><div class="citation">IEEE Trans Med Imaging. 2005 Dec; 24(12):1548-65.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16350916/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT68">Ref list</a>]</p></div><div id="body-link-popper-CIT69" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Comparative assessment of statistical brain MR image segmentation algorithms and their impact on partial volume correction in PET.<div class="authors">Zaidi H, Ruest T, Schoenahl F, Montandon ML</div><div class="citation">Neuroimage. 2006 Oct 1; 32(4):1591-607.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16828315/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT69">Ref list</a>]</p></div><div id="body-link-popper-CIT70" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Multicontext fuzzy clustering for separation of brain tissues in magnetic resonance images.<div class="authors">Zhu C, Jiang T</div><div class="citation">Neuroimage. 2003 Mar; 18(3):685-96.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/12667846/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT70">Ref list</a>]</p></div><div id="body-link-popper-CIT73" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">MRI fuzzy segmentation of brain tissue using neighborhood attraction with neural-network optimization.<div class="authors">Shen S, Sandham W, Granat M, Sterr A</div><div class="citation">IEEE Trans Inf Technol Biomed. 2005 Sep; 9(3):459-67.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16167700/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT73">Ref list</a>]</p></div><div id="body-link-popper-CIT74" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="invert">Review</span> A review of methods for correction of intensity inhomogeneity in MRI.<div class="authors">Vovk U, Pernus F, Likar B</div><div class="citation">IEEE Trans Med Imaging. 2007 Mar; 26(3):405-21.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17354645/" ref="reftype=pubmed&amp;article-id=2825001&amp;issue-id=185087&amp;journal-id=976&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT74">Ref list</a>]</p></div><div id="body-link-popper-CIT72" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">72. <span class="element-citation">Pham DL, Prince JL. An adaptive fuzzy c-means algorithm for image segmentation in the presence of intensity in homogeneities. <span><span class="ref-journal">Pattern Rec Letters. </span>1999;<span class="ref-vol">14</span>:57–68.</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#CIT72">Ref list</a>]</div></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2018-08-30T12:00:25-04:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=ptpmc202&amp;ncbi_phid=F4FC5A18B88138110000000000F100EB&amp;ncbi_session=F4FB7508B28705B1_0206SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2825001%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="./Automated medical image segmentation techniques_files/preloaderWidget.js.download"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov/" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov/" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov/" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov/" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov/">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="https://www.ncbi.nlm.nih.gov/home/about/policies.shtml">Policies and Guidelines</a> | <a href="https://www.ncbi.nlm.nih.gov/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="./Automated medical image segmentation techniques_files/InstrumentPageStarterJS.js.download"> </script>    
    <script type="text/javascript" src="./Automated medical image segmentation techniques_files/hfjs2.js.download"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 63.5938px; left: 490.688px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2825001/#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="top: -100px; left: -100px; display: none;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div><script async="1" src="./Automated medical image segmentation techniques_files/analytics.js.download"></script>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=jmedphys&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC2825001/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- F4FB7508B28705B1_0206SID /projects/PMC/PMCViewer@4.46 ptpmc202 v4.1.r563004 Tue, May 01 2018 03:01:28 -->

<script type="text/javascript" src="./Automated medical image segmentation techniques_files/4065628.js.download" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1" style="display: none;"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><ul class="print-log"><li></li></ul><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-family: MathJax_Size2, sans-serif;"></div></div><div class="ui-ncbipopper-arrow-image ui-ncbipopper-arrow-right" style="display: none; top: -100px; left: -100px; opacity: 1;">&nbsp;</div><div class="ui-ncbipopper-arrow-image ui-ncbipopper-arrow-left" style="display: none; top: -100px; left: -100px; opacity: 1;">&nbsp;</div><div class="ui-ncbipopper-arrow-image ui-ncbipopper-arrow-right" style="display: none; top: -100px; left: -100px; opacity: 1;">&nbsp;</div><div class="ui-ncbipopper-arrow-image ui-ncbipopper-arrow-left" style="display: none; top: -100px; left: -100px; opacity: 1;">&nbsp;</div><div class="ui-ncbipopper-arrow-image ui-ncbipopper-arrow-left" style="display: none; top: -100px; left: -100px; opacity: 1;">&nbsp;</div></body></html>